{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "from neo4j import GraphDatabase\n",
    "from sklearn.manifold import TSNE\n",
    "import numpy as np\n",
    "import altair as alt\n",
    "import pandas as pd\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "driver = GraphDatabase.driver(\"bolt://localhost/PublicOmics\", auth=(\"neo4j\", \"ngs4\"))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    " with driver.session(database=\"neo4j\") as session:\n",
    "    result = session.run(\"\"\"\n",
    "    MATCH (p:Pathway)\n",
    "    RETURN p.Origin AS origin, p.embeddingNode2vec AS embedding, p.Name AS name\n",
    "    \"\"\")\n",
    "    X = pd.DataFrame([dict(record) for record in result])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>origin</th>\n",
       "      <th>embedding</th>\n",
       "      <th>name</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Biocarta</td>\n",
       "      <td>[-0.2407177835702896, -0.07320312410593033, -0...</td>\n",
       "      <td>antisense pathway</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Biocarta</td>\n",
       "      <td>[0.6059083342552185, 0.8333665132522583, 0.592...</td>\n",
       "      <td>hemoglobins chaperone</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Biocarta</td>\n",
       "      <td>[-0.16733059287071228, -0.48236650228500366, 0...</td>\n",
       "      <td>srebp control of lipid synthesis</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Biocarta</td>\n",
       "      <td>[-0.6134916543960571, -0.012800789438188076, 0...</td>\n",
       "      <td>multi-drug resistance factors</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Biocarta</td>\n",
       "      <td>[0.9129302501678467, 0.2537408471107483, 0.645...</td>\n",
       "      <td>inhibition of matrix metalloproteinases</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>735</th>\n",
       "      <td>Kegg</td>\n",
       "      <td>[-0.7549960017204285, 0.816882848739624, 0.963...</td>\n",
       "      <td>basal transcription factors</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>736</th>\n",
       "      <td>Kegg</td>\n",
       "      <td>[-0.12670515477657318, -0.413055419921875, 0.3...</td>\n",
       "      <td>snare interactions in vesicular transport</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>737</th>\n",
       "      <td>Kegg</td>\n",
       "      <td>[-0.5813363194465637, 0.1551935374736786, -0.0...</td>\n",
       "      <td>retinol metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>738</th>\n",
       "      <td>Kegg</td>\n",
       "      <td>[0.04845542460680008, -0.6593214869499207, -0....</td>\n",
       "      <td>lipoic acid metabolism</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>739</th>\n",
       "      <td>Kegg</td>\n",
       "      <td>[0.8294104337692261, -0.041617974638938904, 0....</td>\n",
       "      <td>proteasome</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>740 rows Ã— 3 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "       origin                                          embedding  \\\n",
       "0    Biocarta  [-0.2407177835702896, -0.07320312410593033, -0...   \n",
       "1    Biocarta  [0.6059083342552185, 0.8333665132522583, 0.592...   \n",
       "2    Biocarta  [-0.16733059287071228, -0.48236650228500366, 0...   \n",
       "3    Biocarta  [-0.6134916543960571, -0.012800789438188076, 0...   \n",
       "4    Biocarta  [0.9129302501678467, 0.2537408471107483, 0.645...   \n",
       "..        ...                                                ...   \n",
       "735      Kegg  [-0.7549960017204285, 0.816882848739624, 0.963...   \n",
       "736      Kegg  [-0.12670515477657318, -0.413055419921875, 0.3...   \n",
       "737      Kegg  [-0.5813363194465637, 0.1551935374736786, -0.0...   \n",
       "738      Kegg  [0.04845542460680008, -0.6593214869499207, -0....   \n",
       "739      Kegg  [0.8294104337692261, -0.041617974638938904, 0....   \n",
       "\n",
       "                                          name  \n",
       "0                            antisense pathway  \n",
       "1                        hemoglobins chaperone  \n",
       "2             srebp control of lipid synthesis  \n",
       "3                multi-drug resistance factors  \n",
       "4      inhibition of matrix metalloproteinases  \n",
       "..                                         ...  \n",
       "735                basal transcription factors  \n",
       "736  snare interactions in vesicular transport  \n",
       "737                         retinol metabolism  \n",
       "738                     lipoic acid metabolism  \n",
       "739                                 proteasome  \n",
       "\n",
       "[740 rows x 3 columns]"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "X"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "X_embedded = TSNE(n_components=2, random_state=6).fit_transform(list(X.embedding))\n",
    "\n",
    "df = pd.DataFrame(data = {\n",
    "    \"pathway\": X.name,\n",
    "    \"Source\": X.origin,\n",
    "    \"x\": [value[0] for value in X_embedded],\n",
    "    \"y\": [value[1] for value in X_embedded]\n",
    "})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "<div id=\"altair-viz-07da6cd39f474ca7b6aadfb0417ff1cc\"></div>\n",
       "<script type=\"text/javascript\">\n",
       "  (function(spec, embedOpt){\n",
       "    let outputDiv = document.currentScript.previousElementSibling;\n",
       "    if (outputDiv.id !== \"altair-viz-07da6cd39f474ca7b6aadfb0417ff1cc\") {\n",
       "      outputDiv = document.getElementById(\"altair-viz-07da6cd39f474ca7b6aadfb0417ff1cc\");\n",
       "    }\n",
       "    const paths = {\n",
       "      \"vega\": \"https://cdn.jsdelivr.net/npm//vega@5?noext\",\n",
       "      \"vega-lib\": \"https://cdn.jsdelivr.net/npm//vega-lib?noext\",\n",
       "      \"vega-lite\": \"https://cdn.jsdelivr.net/npm//vega-lite@4.8.1?noext\",\n",
       "      \"vega-embed\": \"https://cdn.jsdelivr.net/npm//vega-embed@6?noext\",\n",
       "    };\n",
       "\n",
       "    function loadScript(lib) {\n",
       "      return new Promise(function(resolve, reject) {\n",
       "        var s = document.createElement('script');\n",
       "        s.src = paths[lib];\n",
       "        s.async = true;\n",
       "        s.onload = () => resolve(paths[lib]);\n",
       "        s.onerror = () => reject(`Error loading script: ${paths[lib]}`);\n",
       "        document.getElementsByTagName(\"head\")[0].appendChild(s);\n",
       "      });\n",
       "    }\n",
       "\n",
       "    function showError(err) {\n",
       "      outputDiv.innerHTML = `<div class=\"error\" style=\"color:red;\">${err}</div>`;\n",
       "      throw err;\n",
       "    }\n",
       "\n",
       "    function displayChart(vegaEmbed) {\n",
       "      vegaEmbed(outputDiv, spec, embedOpt)\n",
       "        .catch(err => showError(`Javascript Error: ${err.message}<br>This usually means there's a typo in your chart specification. See the javascript console for the full traceback.`));\n",
       "    }\n",
       "\n",
       "    if(typeof define === \"function\" && define.amd) {\n",
       "      requirejs.config({paths});\n",
       "      require([\"vega-embed\"], displayChart, err => showError(`Error loading script: ${err.message}`));\n",
       "    } else if (typeof vegaEmbed === \"function\") {\n",
       "      displayChart(vegaEmbed);\n",
       "    } else {\n",
       "      loadScript(\"vega\")\n",
       "        .then(() => loadScript(\"vega-lite\"))\n",
       "        .then(() => loadScript(\"vega-embed\"))\n",
       "        .catch(showError)\n",
       "        .then(() => displayChart(vegaEmbed));\n",
       "    }\n",
       "  })({\"config\": {\"view\": {\"continuousWidth\": 400, \"continuousHeight\": 300}}, \"data\": {\"name\": \"data-4e9182b9b1dc3a1ac4a634cd1527cef6\"}, \"mark\": {\"type\": \"circle\", \"size\": 50}, \"encoding\": {\"color\": {\"type\": \"nominal\", \"field\": \"Source\"}, \"tooltip\": [{\"type\": \"nominal\", \"field\": \"pathway\"}, {\"type\": \"nominal\", \"field\": \"Source\"}, {\"type\": \"quantitative\", \"field\": \"x\"}, {\"type\": \"quantitative\", \"field\": \"y\"}], \"x\": {\"type\": \"quantitative\", \"field\": \"x\"}, \"y\": {\"type\": \"quantitative\", \"field\": \"y\"}}, \"height\": 400, \"width\": 600, \"$schema\": \"https://vega.github.io/schema/vega-lite/v4.8.1.json\", \"datasets\": {\"data-4e9182b9b1dc3a1ac4a634cd1527cef6\": [{\"pathway\": \"antisense pathway\", \"Source\": \"Biocarta\", \"x\": -3.4861435890197754, \"y\": -3.657639265060425}, {\"pathway\": \"hemoglobins chaperone\", \"Source\": \"Biocarta\", \"x\": -11.291159629821777, \"y\": 6.531919002532959}, {\"pathway\": \"srebp control of lipid synthesis\", \"Source\": \"Biocarta\", \"x\": -3.2600057125091553, \"y\": -26.06826400756836}, {\"pathway\": \"multi-drug resistance factors\", \"Source\": \"Biocarta\", \"x\": 20.32229995727539, \"y\": -5.8374128341674805}, {\"pathway\": \"inhibition of matrix metalloproteinases\", \"Source\": \"Biocarta\", \"x\": -4.1424994468688965, \"y\": 10.490935325622559}, {\"pathway\": \"metabolism of anandamide an endogenous cannabinoid\", \"Source\": \"Biocarta\", \"x\": 11.736994743347168, \"y\": 24.09699058532715}, {\"pathway\": \"phospholipase c delta in phospholipid associated cell signaling\", \"Source\": \"Biocarta\", \"x\": -6.173122882843018, \"y\": -21.42814826965332}, {\"pathway\": \"reversal of insulin resistance by leptin\", \"Source\": \"Biocarta\", \"x\": -1.3153691291809082, \"y\": -17.158145904541016}, {\"pathway\": \"eicosanoid metabolism\", \"Source\": \"Biocarta\", \"x\": -11.2048921585083, \"y\": -25.00985336303711}, {\"pathway\": \"mechanism of acetaminophen activity and toxicity\", \"Source\": \"Biocarta\", \"x\": 12.688689231872559, \"y\": 2.0970520973205566}, {\"pathway\": \"hop pathway in cardiac development\", \"Source\": \"Biocarta\", \"x\": -28.492477416992188, \"y\": 3.4214398860931396}, {\"pathway\": \"downregulated of mta-3 in er-negative breast tumors\", \"Source\": \"Biocarta\", \"x\": -4.963634490966797, \"y\": -23.686288833618164}, {\"pathway\": \"the prc2 complex sets long-term gene silencing through modification of histone tails\", \"Source\": \"Biocarta\", \"x\": 4.46491813659668, \"y\": 17.67083740234375}, {\"pathway\": \"mechanisms of transcriptional repression by dna methylation\", \"Source\": \"Biocarta\", \"x\": 15.442728996276855, \"y\": 26.90553092956543}, {\"pathway\": \"mets affect on macrophage differentiation\", \"Source\": \"Biocarta\", \"x\": 8.584458351135254, \"y\": 11.517561912536621}, {\"pathway\": \"role of ran in mitotic spindle regulation\", \"Source\": \"Biocarta\", \"x\": -13.83044147491455, \"y\": 3.1829075813293457}, {\"pathway\": \"mechanism of protein import into the nucleus\", \"Source\": \"Biocarta\", \"x\": -6.528624534606934, \"y\": 12.957788467407227}, {\"pathway\": \"cycling of ran in nucleocytoplasmic transport\", \"Source\": \"Biocarta\", \"x\": 23.092111587524414, \"y\": 15.518963813781738}, {\"pathway\": \"sumoylation by ranbp2 regulates transcriptional repression\", \"Source\": \"Biocarta\", \"x\": -11.458122253417969, \"y\": -3.6079699993133545}, {\"pathway\": \"role of parkin in ubiquitin-proteasomal pathway\", \"Source\": \"Biocarta\", \"x\": -21.490558624267578, \"y\": 9.777351379394531}, {\"pathway\": \"alpha-synuclein and parkin-mediated proteolysis in parkinson`s disease\", \"Source\": \"Biocarta\", \"x\": 18.481693267822266, \"y\": 11.627354621887207}, {\"pathway\": \"er associated degradation (erad) pathway\", \"Source\": \"Biocarta\", \"x\": 15.327347755432129, \"y\": 26.505268096923828}, {\"pathway\": \"basic mechanisms of sumoylation\", \"Source\": \"Biocarta\", \"x\": -12.633354187011719, \"y\": -12.065194129943848}, {\"pathway\": \"regulation of transcriptional activity by pml\", \"Source\": \"Biocarta\", \"x\": 3.966158628463745, \"y\": -24.881502151489258}, {\"pathway\": \"btg family proteins and cell cycle regulation\", \"Source\": \"Biocarta\", \"x\": 14.01663589477539, \"y\": -22.93216323852539}, {\"pathway\": \"p53 signaling pathway\", \"Source\": \"Biocarta\", \"x\": -24.829837799072266, \"y\": 7.6695556640625}, {\"pathway\": \"cyclins and cell cycle regulation\", \"Source\": \"Biocarta\", \"x\": 17.449867248535156, \"y\": 17.203855514526367}, {\"pathway\": \"cell cycle: g1/s check point\", \"Source\": \"Biocarta\", \"x\": -29.263153076171875, \"y\": 4.077281475067139}, {\"pathway\": \"e2f1 destruction pathway\", \"Source\": \"Biocarta\", \"x\": -2.2335336208343506, \"y\": 10.642579078674316}, {\"pathway\": \"cyclin e destruction pathway\", \"Source\": \"Biocarta\", \"x\": 3.755232095718384, \"y\": -3.2814295291900635}, {\"pathway\": \"regulation of p27 phosphorylation during cell cycle progression\", \"Source\": \"Biocarta\", \"x\": 20.46527671813965, \"y\": 18.92486000061035}, {\"pathway\": \"cdk regulation of dna replication\", \"Source\": \"Biocarta\", \"x\": -20.098350524902344, \"y\": -8.658340454101562}, {\"pathway\": \"sonic hedgehog receptor ptc1 regulates cell cycle\", \"Source\": \"Biocarta\", \"x\": 12.272397994995117, \"y\": -5.861385345458984}, {\"pathway\": \"estrogen responsive protein efp controls cell cycle and breast tumors growth\", \"Source\": \"Biocarta\", \"x\": 23.532703399658203, \"y\": 20.046567916870117}, {\"pathway\": \"rb tumor suppressor/checkpoint signaling in response to dna damage\", \"Source\": \"Biocarta\", \"x\": 19.40854263305664, \"y\": -11.228575706481934}, {\"pathway\": \"cdc25 and chk1 regulatory pathway in response to dna damage\", \"Source\": \"Biocarta\", \"x\": -27.842618942260742, \"y\": -3.293541193008423}, {\"pathway\": \"cell cycle: g2/m checkpoint\", \"Source\": \"Biocarta\", \"x\": -13.189069747924805, \"y\": -10.494660377502441}, {\"pathway\": \"atm signaling pathway\", \"Source\": \"Biocarta\", \"x\": -13.510961532592773, \"y\": -13.280234336853027}, {\"pathway\": \"hypoxia and p53 in the cardiovascular system\", \"Source\": \"Biocarta\", \"x\": -20.489883422851562, \"y\": -10.716830253601074}, {\"pathway\": \"tumor suppressor arf inhibits ribosomal biogenesis\", \"Source\": \"Biocarta\", \"x\": -0.13030198216438293, \"y\": 18.316173553466797}, {\"pathway\": \"telomeres telomerase cellular aging and immortality\", \"Source\": \"Biocarta\", \"x\": -1.7002787590026855, \"y\": 2.5546960830688477}, {\"pathway\": \"overview of telomerase protein component gene htert transcriptional regulation\", \"Source\": \"Biocarta\", \"x\": 14.167071342468262, \"y\": -26.109638214111328}, {\"pathway\": \"overview of telomerase rna component gene hterc transcriptional regulation\", \"Source\": \"Biocarta\", \"x\": 13.458556175231934, \"y\": 21.237096786499023}, {\"pathway\": \"role of brca1 brca2 and atr in cancer susceptibility\", \"Source\": \"Biocarta\", \"x\": 6.729744911193848, \"y\": -5.063816070556641}, {\"pathway\": \"brca1 dependent ub ligase activity\", \"Source\": \"Biocarta\", \"x\": -25.09439468383789, \"y\": 8.627852439880371}, {\"pathway\": \"wnt lrp6 signalling\", \"Source\": \"Biocarta\", \"x\": 16.545469284057617, \"y\": -6.772800922393799}, {\"pathway\": \"segmentation clock\", \"Source\": \"Biocarta\", \"x\": -26.186891555786133, \"y\": -1.1379026174545288}, {\"pathway\": \"multi-step regulation of transcription by pitx2\", \"Source\": \"Biocarta\", \"x\": 7.039353847503662, \"y\": 16.537052154541016}, {\"pathway\": \"wnt signaling pathway\", \"Source\": \"Biocarta\", \"x\": -23.0058650970459, \"y\": -8.042903900146484}, {\"pathway\": \"inactivation of gsk3 by akt causes accumulation of b-catenin in alveolar macrophages\", \"Source\": \"Biocarta\", \"x\": -17.72195816040039, \"y\": -5.377535343170166}, {\"pathway\": \"lissencephaly gene (lis1) in neuronal migration and development\", \"Source\": \"Biocarta\", \"x\": 4.2965497970581055, \"y\": 19.20609474182129}, {\"pathway\": \"fosb gene expression and drug abuse\", \"Source\": \"Biocarta\", \"x\": -5.98033332824707, \"y\": 28.438814163208008}, {\"pathway\": \"deregulation of cdk5 in alzheimers disease\", \"Source\": \"Biocarta\", \"x\": -26.529279708862305, \"y\": -7.648880958557129}, {\"pathway\": \"generation of amyloid b-peptide by ps1\", \"Source\": \"Biocarta\", \"x\": 20.8387393951416, \"y\": 7.661555767059326}, {\"pathway\": \"proteolysis and signaling pathway of notch\", \"Source\": \"Biocarta\", \"x\": 21.705547332763672, \"y\": -8.33221435546875}, {\"pathway\": \"g-secretase mediated erbb4 signaling pathway\", \"Source\": \"Biocarta\", \"x\": 4.742188930511475, \"y\": 2.197631597518921}, {\"pathway\": \"neuroregulin receptor degredation protein-1 controls erbb3 receptor recycling\", \"Source\": \"Biocarta\", \"x\": 27.7283878326416, \"y\": -4.377779960632324}, {\"pathway\": \"agrin in postsynaptic differentiation\", \"Source\": \"Biocarta\", \"x\": 2.6716411113739014, \"y\": 17.612125396728516}, {\"pathway\": \"prion pathway\", \"Source\": \"Biocarta\", \"x\": 1.3593459129333496, \"y\": -3.5379607677459717}, {\"pathway\": \"y branching of actin filaments\", \"Source\": \"Biocarta\", \"x\": -4.260114669799805, \"y\": 17.048433303833008}, {\"pathway\": \"how does salmonella hijack a cell\", \"Source\": \"Biocarta\", \"x\": 14.560057640075684, \"y\": -3.8930745124816895}, {\"pathway\": \"role of pi3k subunit p85 in regulation of actin organization and cell migration\", \"Source\": \"Biocarta\", \"x\": 10.731834411621094, \"y\": -11.217456817626953}, {\"pathway\": \"rac1 cell motility signaling pathway\", \"Source\": \"Biocarta\", \"x\": -29.088451385498047, \"y\": -14.030440330505371}, {\"pathway\": \"rho cell motility signaling pathway\", \"Source\": \"Biocarta\", \"x\": -3.2937417030334473, \"y\": 4.753969669342041}, {\"pathway\": \"adp-ribosylation factor\", \"Source\": \"Biocarta\", \"x\": -8.305221557617188, \"y\": 6.286184787750244}, {\"pathway\": \"thrombin signaling and protease-activated receptors\", \"Source\": \"Biocarta\", \"x\": -10.927581787109375, \"y\": -29.424823760986328}, {\"pathway\": \"pkc-catalyzed phosphorylation of inhibitory phosphoprotein of myosin phosphatase\", \"Source\": \"Biocarta\", \"x\": -8.5652437210083, \"y\": 24.695030212402344}, {\"pathway\": \"activation of pkc through g-protein coupled receptors\", \"Source\": \"Biocarta\", \"x\": 8.157807350158691, \"y\": -3.9868581295013428}, {\"pathway\": \"g-protein signaling through tubby proteins\", \"Source\": \"Biocarta\", \"x\": 10.516477584838867, \"y\": -27.274982452392578}, {\"pathway\": \"corticosteroids and cardioprotection\", \"Source\": \"Biocarta\", \"x\": 22.276269912719727, \"y\": -7.045934677124023}, {\"pathway\": \"ion channels and their functional role in vascular endothelium\", \"Source\": \"Biocarta\", \"x\": 3.8730809688568115, \"y\": -20.42270851135254}, {\"pathway\": \"roles of \\u00df arrestin dependent recruitment of src kinases in gpcr signaling\", \"Source\": \"Biocarta\", \"x\": 24.047935485839844, \"y\": -4.17925500869751}, {\"pathway\": \"role of \\u00df-arrestins in the activation and targeting of map kinases\", \"Source\": \"Biocarta\", \"x\": -6.876617431640625, \"y\": -9.852943420410156}, {\"pathway\": \"\\u00df-arrestins in gpcr desensitization\", \"Source\": \"Biocarta\", \"x\": 19.051719665527344, \"y\": -4.974382400512695}, {\"pathway\": \"chrebp regulation by carbohydrates and camp\", \"Source\": \"Biocarta\", \"x\": 0.9699680805206299, \"y\": 26.923709869384766}, {\"pathway\": \"activation of camp-dependent protein kinase pka\", \"Source\": \"Biocarta\", \"x\": 6.957367420196533, \"y\": 15.892650604248047}, {\"pathway\": \"activation of csk by camp-dependent protein kinase inhibits signaling through the t cell receptor\", \"Source\": \"Biocarta\", \"x\": -3.3840959072113037, \"y\": 0.49057087302207947}, {\"pathway\": \"regulation of spermatogenesis by crem\", \"Source\": \"Biocarta\", \"x\": -10.143387794494629, \"y\": 7.113526821136475}, {\"pathway\": \"rho-selective guanine exchange factor akap13 mediates stress fiber formation\", \"Source\": \"Biocarta\", \"x\": -14.501511573791504, \"y\": 11.604801177978516}, {\"pathway\": \"protein kinase a at the centrosome\", \"Source\": \"Biocarta\", \"x\": 22.85384750366211, \"y\": 8.398908615112305}, {\"pathway\": \"akap95 role in mitosis and chromosome dynamics\", \"Source\": \"Biocarta\", \"x\": -11.021953582763672, \"y\": 22.167253494262695}, {\"pathway\": \"stathmin and breast cancer resistance to antimicrotubule agents\", \"Source\": \"Biocarta\", \"x\": 19.947959899902344, \"y\": -1.156866431236267}, {\"pathway\": \"nitric oxide signaling pathway\", \"Source\": \"Biocarta\", \"x\": -16.468318939208984, \"y\": 20.85157585144043}, {\"pathway\": \"signaling pathway from g-protein families\", \"Source\": \"Biocarta\", \"x\": 3.463596820831299, \"y\": -0.6106978058815002}, {\"pathway\": \"attenuation of gpcr signaling\", \"Source\": \"Biocarta\", \"x\": -29.853837966918945, \"y\": 4.263387203216553}, {\"pathway\": \"regulation of ck1/cdk5 by type 1 glutamate receptors\", \"Source\": \"Biocarta\", \"x\": -4.777199745178223, \"y\": 23.324264526367188}, {\"pathway\": \"how progesterone initiates the oocyte maturation\", \"Source\": \"Biocarta\", \"x\": 8.325453758239746, \"y\": 21.19045066833496}, {\"pathway\": \"cystic fibrosis transmembrane conductance regulator (cftr) and beta 2 adrenergic receptor (b2ar) pathway\", \"Source\": \"Biocarta\", \"x\": 19.37238311767578, \"y\": 22.931333541870117}, {\"pathway\": \"phospholipase c-epsilon pathway\", \"Source\": \"Biocarta\", \"x\": -24.863910675048828, \"y\": -3.3667407035827637}, {\"pathway\": \"gata3 participate in activating the th2 cytokine genes expression\", \"Source\": \"Biocarta\", \"x\": -9.93896484375, \"y\": 18.169044494628906}, {\"pathway\": \"repression of pain sensation by the transcriptional regulator dream\", \"Source\": \"Biocarta\", \"x\": -8.012393951416016, \"y\": -11.584857940673828}, {\"pathway\": \"transcription regulation by methyltransferase of carm1\", \"Source\": \"Biocarta\", \"x\": -9.774785041809082, \"y\": -5.1595940589904785}, {\"pathway\": \"regulation of bad phosphorylation\", \"Source\": \"Biocarta\", \"x\": -5.193638324737549, \"y\": 17.49783706665039}, {\"pathway\": \"control of skeletal myogenesis by hdac and calcium/calmodulin-dependent kinase (camk)\", \"Source\": \"Biocarta\", \"x\": -9.546089172363281, \"y\": -21.708927154541016}, {\"pathway\": \"signal dependent regulation of myogenesis by corepressor mitr\", \"Source\": \"Biocarta\", \"x\": 5.311272621154785, \"y\": 2.2874860763549805}, {\"pathway\": \"regulation of pgc-1a\", \"Source\": \"Biocarta\", \"x\": -0.7016587257385254, \"y\": -23.810359954833984}, {\"pathway\": \"ca-calmodulin-dependent protein kinase activation\", \"Source\": \"Biocarta\", \"x\": 14.860844612121582, \"y\": 14.234746932983398}, {\"pathway\": \"nfat and hypertrophy of the heart \", \"Source\": \"Biocarta\", \"x\": -17.673564910888672, \"y\": 19.542949676513672}, {\"pathway\": \"endocytotic role of ndk phosphins and dynamin\", \"Source\": \"Biocarta\", \"x\": -1.3059741258621216, \"y\": -21.038667678833008}, {\"pathway\": \"effects of calcineurin in keratinocyte differentiation\", \"Source\": \"Biocarta\", \"x\": -2.6654319763183594, \"y\": -24.824024200439453}, {\"pathway\": \"ionomycin and phorbal ester signaling pathway\", \"Source\": \"Biocarta\", \"x\": 12.730476379394531, \"y\": -8.412534713745117}, {\"pathway\": \"cbl mediated ligand-induced downregulation of egf receptors pathway\", \"Source\": \"Biocarta\", \"x\": -30.59311294555664, \"y\": -12.112870216369629}, {\"pathway\": \"oxidative stress induced gene expression via nrf2\", \"Source\": \"Biocarta\", \"x\": -6.65777063369751, \"y\": -23.938600540161133}, {\"pathway\": \"mcalpain and friends in cell motility\", \"Source\": \"Biocarta\", \"x\": -29.67679214477539, \"y\": -13.289579391479492}, {\"pathway\": \"ucalpain and friends in cell spread\", \"Source\": \"Biocarta\", \"x\": 12.350640296936035, \"y\": 13.424210548400879}, {\"pathway\": \"erk and pi-3 kinase are necessary for collagen binding in corneal epithelia\", \"Source\": \"Biocarta\", \"x\": -14.776494979858398, \"y\": -15.862113952636719}, {\"pathway\": \"integrin signaling pathway\", \"Source\": \"Biocarta\", \"x\": 10.953133583068848, \"y\": 11.842426300048828}, {\"pathway\": \"cell to cell adhesion signaling\", \"Source\": \"Biocarta\", \"x\": 15.480023384094238, \"y\": 9.24042797088623}, {\"pathway\": \"vegf hypoxia and angiogenesis\", \"Source\": \"Biocarta\", \"x\": 24.043027877807617, \"y\": -7.192504405975342}, {\"pathway\": \"actions of nitric oxide in the heart\", \"Source\": \"Biocarta\", \"x\": 1.3239232301712036, \"y\": -7.7049336433410645}, {\"pathway\": \"pertussis toxin-insensitive ccr5 signaling in macrophage\", \"Source\": \"Biocarta\", \"x\": 12.013103485107422, \"y\": 25.16288185119629}, {\"pathway\": \"cxcr4 signaling pathway\", \"Source\": \"Biocarta\", \"x\": -12.966619491577148, \"y\": 14.974379539489746}, {\"pathway\": \"phospholipase c signaling pathway\", \"Source\": \"Biocarta\", \"x\": 20.528919219970703, \"y\": -5.070228099822998}, {\"pathway\": \"b cell survival pathway\", \"Source\": \"Biocarta\", \"x\": 1.631789207458496, \"y\": -11.10525894165039}, {\"pathway\": \"mtor signaling pathway\", \"Source\": \"Biocarta\", \"x\": 16.590307235717773, \"y\": -20.851478576660156}, {\"pathway\": \"regulation of eif-4e and p70s6 kinase\", \"Source\": \"Biocarta\", \"x\": -9.845115661621094, \"y\": 4.6907639503479}, {\"pathway\": \"skeletal muscle hypertrophy is regulated via akt-mtor pathway\", \"Source\": \"Biocarta\", \"x\": -22.69968032836914, \"y\": -0.0428120419383049}, {\"pathway\": \"role of nicotinic acetylcholine receptors in the regulation of apoptosis\", \"Source\": \"Biocarta\", \"x\": 18.36469268798828, \"y\": 12.395943641662598}, {\"pathway\": \"akt signaling pathway\", \"Source\": \"Biocarta\", \"x\": -25.229951858520508, \"y\": -12.427783966064453}, {\"pathway\": \"ras signaling pathway\", \"Source\": \"Biocarta\", \"x\": 16.74648666381836, \"y\": -21.405302047729492}, {\"pathway\": \"phosphoinositides and their downstream targets\", \"Source\": \"Biocarta\", \"x\": 12.72800064086914, \"y\": -11.179059982299805}, {\"pathway\": \"ras-independent pathway in nk cell-mediated cytotoxicity\", \"Source\": \"Biocarta\", \"x\": 10.070920944213867, \"y\": -6.625293254852295}, {\"pathway\": \"human cytomegalovirus and map kinase pathways\", \"Source\": \"Biocarta\", \"x\": -10.325133323669434, \"y\": 25.79123306274414}, {\"pathway\": \"influence of ras and rho proteins on g1 to s transition\", \"Source\": \"Biocarta\", \"x\": -11.542961120605469, \"y\": -9.609090805053711}, {\"pathway\": \"trefoil factors initiate mucosal healing\", \"Source\": \"Biocarta\", \"x\": 23.172338485717773, \"y\": 10.123947143554688}, {\"pathway\": \"keratinocyte differentiation\", \"Source\": \"Biocarta\", \"x\": -10.370627403259277, \"y\": -27.29680633544922}, {\"pathway\": \"role of egf receptor transactivation by gpcrs in cardiac hypertrophy\", \"Source\": \"Biocarta\", \"x\": 3.871614933013916, \"y\": -17.915260314941406}, {\"pathway\": \"cadmium induces dna synthesis and proliferation in macrophages\", \"Source\": \"Biocarta\", \"x\": 9.310739517211914, \"y\": -1.345755934715271}, {\"pathway\": \"fmlp induced chemokine gene expression in hmc-1 cells\", \"Source\": \"Biocarta\", \"x\": 27.073856353759766, \"y\": -11.3961763381958}, {\"pathway\": \"aspirin blocks signaling pathway involved in platelet activation\", \"Source\": \"Biocarta\", \"x\": -8.247993469238281, \"y\": -29.297096252441406}, {\"pathway\": \"ccr3 signaling in eosinophils\", \"Source\": \"Biocarta\", \"x\": -7.569040298461914, \"y\": 4.601357460021973}, {\"pathway\": \"role of mal in rho-mediated activation of srf\", \"Source\": \"Biocarta\", \"x\": -16.493669509887695, \"y\": 0.5377597212791443}, {\"pathway\": \"anthrax toxin mechanism of action\", \"Source\": \"Biocarta\", \"x\": 5.3129754066467285, \"y\": -19.82500457763672}, {\"pathway\": \"melanocyte development and pigmentation pathway\", \"Source\": \"Biocarta\", \"x\": -7.388789176940918, \"y\": -29.9857120513916}, {\"pathway\": \"regulation of splicing through sam68\", \"Source\": \"Biocarta\", \"x\": -20.43146514892578, \"y\": 8.344433784484863}, {\"pathway\": \"sprouty regulation of tyrosine kinase signals\", \"Source\": \"Biocarta\", \"x\": 4.426208019256592, \"y\": 13.335334777832031}, {\"pathway\": \"phosphorylation of mek1 by cdk5/p35 down regulates the map kinase pathway\", \"Source\": \"Biocarta\", \"x\": -11.261402130126953, \"y\": -1.8122690916061401}, {\"pathway\": \"erk1/erk2 mapk signaling pathway\", \"Source\": \"Biocarta\", \"x\": -25.22175407409668, \"y\": 5.300680160522461}, {\"pathway\": \"stat3 signaling pathway\", \"Source\": \"Biocarta\", \"x\": -7.10255765914917, \"y\": 15.905525207519531}, {\"pathway\": \"il 6 signaling pathway\", \"Source\": \"Biocarta\", \"x\": -24.18924331665039, \"y\": -0.30757367610931396}, {\"pathway\": \"role of erbb2 in signal transduction and oncology\", \"Source\": \"Biocarta\", \"x\": 11.980570793151855, \"y\": 1.3548035621643066}, {\"pathway\": \"il-2 receptor beta chain in t cell activation\", \"Source\": \"Biocarta\", \"x\": 15.676742553710938, \"y\": 16.98853302001953}, {\"pathway\": \"il 2 signaling pathway\", \"Source\": \"Biocarta\", \"x\": 22.22904396057129, \"y\": 12.07345962524414}, {\"pathway\": \"il 3 signaling pathway\", \"Source\": \"Biocarta\", \"x\": 0.603820264339447, \"y\": 19.530689239501953}, {\"pathway\": \"epo signaling pathway\", \"Source\": \"Biocarta\", \"x\": -26.915348052978516, \"y\": -3.0731759071350098}, {\"pathway\": \"growth hormone signaling pathway\", \"Source\": \"Biocarta\", \"x\": -3.3764138221740723, \"y\": -20.666950225830078}, {\"pathway\": \"insulin signaling pathway\", \"Source\": \"Biocarta\", \"x\": -2.8073291778564453, \"y\": -1.0274083614349365}, {\"pathway\": \"multiple antiapoptotic pathways from igf-1r signaling lead to bad phosphorylation\", \"Source\": \"Biocarta\", \"x\": -11.991994857788086, \"y\": -10.222661972045898}, {\"pathway\": \"the igf-1 receptor and longevity\", \"Source\": \"Biocarta\", \"x\": 21.896995544433594, \"y\": 5.951839447021484}, {\"pathway\": \"pten dependent cell cycle arrest and apoptosis\", \"Source\": \"Biocarta\", \"x\": -26.20409393310547, \"y\": 5.423798561096191}, {\"pathway\": \"phospholipids as signalling intermediaries\", \"Source\": \"Biocarta\", \"x\": -14.726968765258789, \"y\": 25.895357131958008}, {\"pathway\": \"role of erk5 in neuronal survival pathway\", \"Source\": \"Biocarta\", \"x\": -10.156065940856934, \"y\": -21.894424438476562}, {\"pathway\": \"trka receptor signaling pathway\", \"Source\": \"Biocarta\", \"x\": 11.220592498779297, \"y\": -4.124739646911621}, {\"pathway\": \"transcription factor creb and its extracellular signals\", \"Source\": \"Biocarta\", \"x\": -27.055997848510742, \"y\": -15.499401092529297}, {\"pathway\": \"bioactive peptide induced signaling pathway\", \"Source\": \"Biocarta\", \"x\": -12.916518211364746, \"y\": -11.566375732421875}, {\"pathway\": \"links between pyk2 and map kinases\", \"Source\": \"Biocarta\", \"x\": -13.310568809509277, \"y\": -25.274646759033203}, {\"pathway\": \"angiotensin ii mediated activation of jnk pathway via pyk2 dependent signaling\", \"Source\": \"Biocarta\", \"x\": -7.652091979980469, \"y\": -30.847932815551758}, {\"pathway\": \"calcium signaling by hbx of hepatitis b virus\", \"Source\": \"Biocarta\", \"x\": 24.612178802490234, \"y\": -7.431556701660156}, {\"pathway\": \"nerve growth factor pathway (ngf)\", \"Source\": \"Biocarta\", \"x\": 11.694570541381836, \"y\": -5.264969825744629}, {\"pathway\": \"igf-1 signaling pathway\", \"Source\": \"Biocarta\", \"x\": -29.376476287841797, \"y\": -6.540802478790283}, {\"pathway\": \"pdgf signaling pathway\", \"Source\": \"Biocarta\", \"x\": -4.320044994354248, \"y\": 15.779609680175781}, {\"pathway\": \"egf signaling pathway\", \"Source\": \"Biocarta\", \"x\": 12.902613639831543, \"y\": -26.7821102142334}, {\"pathway\": \"tpo signaling pathway\", \"Source\": \"Biocarta\", \"x\": -22.182430267333984, \"y\": -12.197641372680664}, {\"pathway\": \"inhibition of cellular proliferation by gleevec\", \"Source\": \"Biocarta\", \"x\": 13.735610008239746, \"y\": -11.721719741821289}, {\"pathway\": \"fc epsilon receptor i signaling in mast cells\", \"Source\": \"Biocarta\", \"x\": 4.223362445831299, \"y\": -1.8884837627410889}, {\"pathway\": \"bcr signaling pathway\", \"Source\": \"Biocarta\", \"x\": -10.483341217041016, \"y\": -3.7425427436828613}, {\"pathway\": \"t cell receptor signaling pathway\", \"Source\": \"Biocarta\", \"x\": 20.408010482788086, \"y\": 7.41601037979126}, {\"pathway\": \"role of mef2d in t-cell apoptosis\", \"Source\": \"Biocarta\", \"x\": 11.437823295593262, \"y\": 10.672648429870605}, {\"pathway\": \"lck and fyn tyrosine kinases in initiation of tcr activation\", \"Source\": \"Biocarta\", \"x\": 20.609121322631836, \"y\": 13.092133522033691}, {\"pathway\": \"the co-stimulatory signal during t-cell activation\", \"Source\": \"Biocarta\", \"x\": -9.05738353729248, \"y\": -23.8112735748291}, {\"pathway\": \"il 4 signaling pathway\", \"Source\": \"Biocarta\", \"x\": 8.821969985961914, \"y\": -18.65907096862793}, {\"pathway\": \"antigen processing and presentation\", \"Source\": \"Biocarta\", \"x\": -11.095619201660156, \"y\": -9.850906372070312}, {\"pathway\": \"rna polymerase iii transcription\", \"Source\": \"Biocarta\", \"x\": 24.78839111328125, \"y\": -6.170398235321045}, {\"pathway\": \"visceral fat deposits and the metabolic syndrome\", \"Source\": \"Biocarta\", \"x\": -6.0842719078063965, \"y\": 27.842933654785156}, {\"pathway\": \"basic mechanism of action of ppara pparb(d) and pparg and effects on gene expression\", \"Source\": \"Biocarta\", \"x\": -15.483214378356934, \"y\": -23.412700653076172}, {\"pathway\": \"degradation of the rar and rxr by the proteasome\", \"Source\": \"Biocarta\", \"x\": -14.293785095214844, \"y\": 20.00421714782715}, {\"pathway\": \"map kinase inactivation of smrt corepressor\", \"Source\": \"Biocarta\", \"x\": 15.45639419555664, \"y\": 18.140531539916992}, {\"pathway\": \"nuclear receptors coordinate the activities of chromatin remodeling complexes and coactivators to facilitate initiation of transcription in carcinoma cells\", \"Source\": \"Biocarta\", \"x\": 20.790817260742188, \"y\": 4.640673637390137}, {\"pathway\": \"carm1 and regulation of the estrogen receptor\", \"Source\": \"Biocarta\", \"x\": -10.132712364196777, \"y\": -26.194658279418945}, {\"pathway\": \"chromatin remodeling by hswi/snf atp-dependent complexes\", \"Source\": \"Biocarta\", \"x\": 21.277332305908203, \"y\": 0.6608631610870361}, {\"pathway\": \"the information processing pathway at the ifn beta enhancer\", \"Source\": \"Biocarta\", \"x\": -3.866480827331543, \"y\": 4.492013931274414}, {\"pathway\": \"control of gene expression by vitamin d receptor\", \"Source\": \"Biocarta\", \"x\": 10.12961196899414, \"y\": 1.7417659759521484}, {\"pathway\": \"role of ppar-gamma coactivators in obesity and thermogenesis\", \"Source\": \"Biocarta\", \"x\": 10.317473411560059, \"y\": -1.5475685596466064}, {\"pathway\": \"mechanism of gene regulation by peroxisome proliferators via ppara\", \"Source\": \"Biocarta\", \"x\": 10.56238842010498, \"y\": 23.876502990722656}, {\"pathway\": \"inhibition of huntingtons disease neurodegeneration by histone deacetylase inhibitors\", \"Source\": \"Biocarta\", \"x\": -4.182599067687988, \"y\": -10.102632522583008}, {\"pathway\": \"il-7 signal transduction\", \"Source\": \"Biocarta\", \"x\": -6.738153457641602, \"y\": -5.19857120513916}, {\"pathway\": \"pelp1 modulation of estrogen receptor activity\", \"Source\": \"Biocarta\", \"x\": -22.15488052368164, \"y\": -1.3761224746704102}, {\"pathway\": \"acetylation and deacetylation of rela in nucleus\", \"Source\": \"Biocarta\", \"x\": 1.300719976425171, \"y\": 9.11954402923584}, {\"pathway\": \"erythropoietin mediated neuroprotection through nf-kb\", \"Source\": \"Biocarta\", \"x\": -14.878209114074707, \"y\": 10.196029663085938}, {\"pathway\": \"hypoxia-inducible factor in the cardivascular system\", \"Source\": \"Biocarta\", \"x\": 9.57265567779541, \"y\": -2.5896520614624023}, {\"pathway\": \"ahr signal transduction pathway\", \"Source\": \"Biocarta\", \"x\": -30.42961311340332, \"y\": -13.901756286621094}, {\"pathway\": \"opposing roles of aif in apoptosis and cell survival\", \"Source\": \"Biocarta\", \"x\": 5.920709609985352, \"y\": 18.992952346801758}, {\"pathway\": \"role of mitochondria in apoptotic signaling\", \"Source\": \"Biocarta\", \"x\": -11.795475959777832, \"y\": 11.561431884765625}, {\"pathway\": \"apoptotic signaling in response to dna damage\", \"Source\": \"Biocarta\", \"x\": 16.812795639038086, \"y\": 7.637640476226807}, {\"pathway\": \"regulation of cell cycle progression by plk3\", \"Source\": \"Biocarta\", \"x\": 7.577765941619873, \"y\": -20.110424041748047}, {\"pathway\": \"west nile virus\", \"Source\": \"Biocarta\", \"x\": -1.0991437435150146, \"y\": 26.2640438079834}, {\"pathway\": \"stress induction of hsp regulation\", \"Source\": \"Biocarta\", \"x\": -7.670477390289307, \"y\": 25.112089157104492}, {\"pathway\": \"fas signaling pathway (cd95)\", \"Source\": \"Biocarta\", \"x\": -19.384380340576172, \"y\": -16.579683303833008}, {\"pathway\": \"hiv-1 nef: negative effector of fas and tnf\", \"Source\": \"Biocarta\", \"x\": 2.3793914318084717, \"y\": 1.0513129234313965}, {\"pathway\": \"caspase cascade in apoptosis\", \"Source\": \"Biocarta\", \"x\": -10.175275802612305, \"y\": 24.986928939819336}, {\"pathway\": \"induction of apoptosis through dr3 and dr4/5 death receptors\", \"Source\": \"Biocarta\", \"x\": -21.97449493408203, \"y\": -2.6289305686950684}, {\"pathway\": \"tnfr1 signaling pathway\", \"Source\": \"Biocarta\", \"x\": -15.655051231384277, \"y\": -6.557297706604004}, {\"pathway\": \"sodd/tnfr1 signaling pathway\", \"Source\": \"Biocarta\", \"x\": -8.90983772277832, \"y\": -16.54669952392578}, {\"pathway\": \"ceramide signaling pathway\", \"Source\": \"Biocarta\", \"x\": 1.8861234188079834, \"y\": -1.8830852508544922}, {\"pathway\": \"tnf/stress related signaling\", \"Source\": \"Biocarta\", \"x\": -12.722001075744629, \"y\": -1.5641661882400513}, {\"pathway\": \"d4gdi signaling pathway\", \"Source\": \"Biocarta\", \"x\": -0.3464602828025818, \"y\": -20.013599395751953}, {\"pathway\": \"tsp-1 induced apoptosis in microvascular endothelial cell\", \"Source\": \"Biocarta\", \"x\": -4.859819412231445, \"y\": -3.476576089859009}, {\"pathway\": \"granzyme a mediated apoptosis pathway\", \"Source\": \"Biocarta\", \"x\": 5.376379489898682, \"y\": -14.939130783081055}, {\"pathway\": \"apoptotic dna-fragmentation and tissue homeostasis\", \"Source\": \"Biocarta\", \"x\": 22.634479522705078, \"y\": 19.804597854614258}, {\"pathway\": \"internal ribosome entry pathway\", \"Source\": \"Biocarta\", \"x\": -5.0658860206604, \"y\": 5.536052703857422}, {\"pathway\": \"eukaryotic protein translation\", \"Source\": \"Biocarta\", \"x\": -26.602630615234375, \"y\": -8.891937255859375}, {\"pathway\": \"regulation of eif2\", \"Source\": \"Biocarta\", \"x\": 9.49499797821045, \"y\": -7.3769145011901855}, {\"pathway\": \"double stranded rna induced gene expression\", \"Source\": \"Biocarta\", \"x\": -4.219362258911133, \"y\": 17.845212936401367}, {\"pathway\": \"bone remodeling\", \"Source\": \"Biocarta\", \"x\": -8.241033554077148, \"y\": -22.136402130126953}, {\"pathway\": \"toll-like receptor pathway\", \"Source\": \"Biocarta\", \"x\": -1.8747944831848145, \"y\": 26.741409301757812}, {\"pathway\": \"signal transduction through il1r\", \"Source\": \"Biocarta\", \"x\": -5.9441633224487305, \"y\": -9.889334678649902}, {\"pathway\": \"nf-kb signaling pathway\", \"Source\": \"Biocarta\", \"x\": -5.659682750701904, \"y\": 12.876214981079102}, {\"pathway\": \"cd40l signaling pathway\", \"Source\": \"Biocarta\", \"x\": 5.130429744720459, \"y\": -5.008100509643555}, {\"pathway\": \"tnfr2 signaling pathway\", \"Source\": \"Biocarta\", \"x\": 16.473329544067383, \"y\": 22.923992156982422}, {\"pathway\": \"yaci and bcma stimulation of b cell immune responses\", \"Source\": \"Biocarta\", \"x\": 5.618435382843018, \"y\": 20.487279891967773}, {\"pathway\": \"ctcf: first multivalent nuclear factor\", \"Source\": \"Biocarta\", \"x\": -18.81023406982422, \"y\": -12.564929008483887}, {\"pathway\": \"alk in cardiac myocytes\", \"Source\": \"Biocarta\", \"x\": 24.954952239990234, \"y\": 7.456628799438477}, {\"pathway\": \"tgf beta signaling pathway\", \"Source\": \"Biocarta\", \"x\": 23.856380462646484, \"y\": 10.088584899902344}, {\"pathway\": \"nfkb activation by nontypeable hemophilus influenzae\", \"Source\": \"Biocarta\", \"x\": -14.188493728637695, \"y\": -29.10605239868164}, {\"pathway\": \"regulation of map kinase pathways through dual specificity phosphatases\", \"Source\": \"Biocarta\", \"x\": -1.3214337825775146, \"y\": -13.709351539611816}, {\"pathway\": \"mapkinase signaling pathway\", \"Source\": \"Biocarta\", \"x\": -20.795040130615234, \"y\": -14.89862060546875}, {\"pathway\": \"p38 mapk signaling pathway\", \"Source\": \"Biocarta\", \"x\": -7.2621684074401855, \"y\": 2.370002031326294}, {\"pathway\": \"il-10 anti-inflammatory signaling pathway\", \"Source\": \"Biocarta\", \"x\": -1.0108497142791748, \"y\": 16.382699966430664}, {\"pathway\": \"the 41bb-dependent immune response\", \"Source\": \"Biocarta\", \"x\": -2.6889729499816895, \"y\": -6.287875652313232}, {\"pathway\": \"il12 and stat4 dependent signaling pathway in th1 development\", \"Source\": \"Biocarta\", \"x\": 4.812703609466553, \"y\": -20.517654418945312}, {\"pathway\": \"no2-dependent il-12 pathway in nk cells\", \"Source\": \"Biocarta\", \"x\": -0.5673454999923706, \"y\": 21.713167190551758}, {\"pathway\": \"chaperones modulate interferon signaling pathway\", \"Source\": \"Biocarta\", \"x\": 2.4060585498809814, \"y\": 22.60845947265625}, {\"pathway\": \"ifn gamma signaling pathway\", \"Source\": \"Biocarta\", \"x\": 1.9598149061203003, \"y\": -17.58531951904297}, {\"pathway\": \"ifn alpha signaling pathway\", \"Source\": \"Biocarta\", \"x\": -13.649641036987305, \"y\": 8.193668365478516}, {\"pathway\": \"il22 soluble receptor signaling pathway\", \"Source\": \"Biocarta\", \"x\": -8.333731651306152, \"y\": 25.9808349609375}, {\"pathway\": \"visual signal transduction\", \"Source\": \"Biocarta\", \"x\": 14.693329811096191, \"y\": -8.537662506103516}, {\"pathway\": \"hiv-1 defeats host-mediated resistance by cem15\", \"Source\": \"Biocarta\", \"x\": -0.17676730453968048, \"y\": -23.58531379699707}, {\"pathway\": \"regulators of bone mineralization\", \"Source\": \"Biocarta\", \"x\": 7.861416816711426, \"y\": -17.938793182373047}, {\"pathway\": \"platelet amyloid precursor protein pathway\", \"Source\": \"Biocarta\", \"x\": -9.89815616607666, \"y\": 19.713064193725586}, {\"pathway\": \"intrinsic prothrombin activation pathway\", \"Source\": \"Biocarta\", \"x\": -16.18536949157715, \"y\": 3.6637518405914307}, {\"pathway\": \"extrinsic prothrombin activation pathway\", \"Source\": \"Biocarta\", \"x\": 24.70077133178711, \"y\": -3.9147677421569824}, {\"pathway\": \"fibrinolysis pathway\", \"Source\": \"Biocarta\", \"x\": -7.926388263702393, \"y\": -22.940462112426758}, {\"pathway\": \"transcriptional activation of dbpb from mrna\", \"Source\": \"Biocarta\", \"x\": 7.364354133605957, \"y\": -23.724468231201172}, {\"pathway\": \"proteasome complex\", \"Source\": \"Biocarta\", \"x\": -25.719579696655273, \"y\": -7.556016445159912}, {\"pathway\": \"sumoylation as a mechanism to modulate ctbp-dependent gene responses\", \"Source\": \"Biocarta\", \"x\": 3.70790958404541, \"y\": 28.752424240112305}, {\"pathway\": \"gamma-aminobutyric acid receptor life cycle pathway\", \"Source\": \"Biocarta\", \"x\": 1.9580073356628418, \"y\": 9.799881935119629}, {\"pathway\": \"cardiac protection against ros\", \"Source\": \"Biocarta\", \"x\": 5.166711330413818, \"y\": 25.201587677001953}, {\"pathway\": \"polyadenylation of mrna\", \"Source\": \"Biocarta\", \"x\": -1.0107187032699585, \"y\": 4.4095234870910645}, {\"pathway\": \"spliceosomal assembly\", \"Source\": \"Biocarta\", \"x\": 23.10028648376465, \"y\": 7.003201961517334}, {\"pathway\": \"phosphatidylcholine biosynthesis pathway\", \"Source\": \"Biocarta\", \"x\": -9.955270767211914, \"y\": -25.072147369384766}, {\"pathway\": \"classical complement pathway\", \"Source\": \"Biocarta\", \"x\": 5.853876113891602, \"y\": -23.604148864746094}, {\"pathway\": \"lectin induced complement pathway\", \"Source\": \"Biocarta\", \"x\": 13.32693099975586, \"y\": -24.13747215270996}, {\"pathway\": \"alternative complement pathway\", \"Source\": \"Biocarta\", \"x\": -13.69694995880127, \"y\": 20.02503776550293}, {\"pathway\": \"dicer pathway\", \"Source\": \"Biocarta\", \"x\": -3.7617316246032715, \"y\": -21.452281951904297}, {\"pathway\": \"urea cycle\", \"Source\": \"HumanCyc\", \"x\": 12.598915100097656, \"y\": -25.827058792114258}, {\"pathway\": \"citrulline-nitric oxide cycle\", \"Source\": \"HumanCyc\", \"x\": 19.06201171875, \"y\": -8.19573974609375}, {\"pathway\": \"alpha-tocopherol degradation\", \"Source\": \"HumanCyc\", \"x\": -6.932648181915283, \"y\": -21.39411735534668}, {\"pathway\": \"phosphatidylserine biosynthesis I\", \"Source\": \"HumanCyc\", \"x\": 13.062461853027344, \"y\": -23.326601028442383}, {\"pathway\": \"phosphatidylcholine biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 15.473416328430176, \"y\": 25.4344425201416}, {\"pathway\": \"phosphatidylethanolamine biosynthesis II\", \"Source\": \"HumanCyc\", \"x\": 30.00768280029297, \"y\": -9.470712661743164}, {\"pathway\": \"flavin biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 23.367708206176758, \"y\": 18.43701171875}, {\"pathway\": \"phenylalanine degradation/tyrosine biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 22.795114517211914, \"y\": 18.173938751220703}, {\"pathway\": \"pentose phosphate pathway (non-oxidative branch)\", \"Source\": \"HumanCyc\", \"x\": 9.941458702087402, \"y\": -12.048995971679688}, {\"pathway\": \"pentose phosphate pathway\", \"Source\": \"HumanCyc\", \"x\": 2.5007174015045166, \"y\": -21.869779586791992}, {\"pathway\": \"pentose phosphate pathway (oxidative branch)\", \"Source\": \"HumanCyc\", \"x\": -4.0941314697265625, \"y\": -28.25233268737793}, {\"pathway\": \"oxidized GTP and dGTP detoxification\", \"Source\": \"HumanCyc\", \"x\": 21.24779510498047, \"y\": 1.6634700298309326}, {\"pathway\": \"anandamide degradation\", \"Source\": \"HumanCyc\", \"x\": -23.53097152709961, \"y\": -9.831613540649414}, {\"pathway\": \"L-glutamine tRNA biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -7.071475028991699, \"y\": -14.10655689239502}, {\"pathway\": \"hypusine biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -0.11644334346055984, \"y\": 22.382768630981445}, {\"pathway\": \"mitochondrial L-carnitine shuttle\", \"Source\": \"HumanCyc\", \"x\": 2.7916178703308105, \"y\": 24.70811653137207}, {\"pathway\": \"D-glucuronate degradation\", \"Source\": \"HumanCyc\", \"x\": 2.663626194000244, \"y\": -16.72664451599121}, {\"pathway\": \"methylglyoxal degradation I\", \"Source\": \"HumanCyc\", \"x\": -2.2179152965545654, \"y\": 8.432219505310059}, {\"pathway\": \"methylglyoxal degradation VI\", \"Source\": \"HumanCyc\", \"x\": -8.951830863952637, \"y\": -4.046635627746582}, {\"pathway\": \"lactate fermentation (reoxidation of cytosolic NADH)\", \"Source\": \"HumanCyc\", \"x\": -6.602660179138184, \"y\": -5.41062068939209}, {\"pathway\": \"7-(3-amino-3-carboxypropyl)-wyosine biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 2.2078521251678467, \"y\": -18.94978141784668}, {\"pathway\": \"wybutosine biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -3.5426249504089355, \"y\": 26.276281356811523}, {\"pathway\": \"epoxysqualene biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 11.882341384887695, \"y\": 3.776278495788574}, {\"pathway\": \"superpathway of cholesterol biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -14.414131164550781, \"y\": -12.897823333740234}, {\"pathway\": \"cholesterol biosynthesis II (via 24,25-dihydrolanosterol)\", \"Source\": \"HumanCyc\", \"x\": 12.554289817810059, \"y\": -27.293651580810547}, {\"pathway\": \"cholesterol biosynthesis III (via desmosterol)\", \"Source\": \"HumanCyc\", \"x\": 1.9919483661651611, \"y\": -9.426186561584473}, {\"pathway\": \"cholesterol biosynthesis I\", \"Source\": \"HumanCyc\", \"x\": 1.1175978183746338, \"y\": 17.893484115600586}, {\"pathway\": \"zymosterol biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 1.9189025163650513, \"y\": 14.380572319030762}, {\"pathway\": \"lanosterol biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 15.245074272155762, \"y\": 7.140285015106201}, {\"pathway\": \"ketogenesis\", \"Source\": \"HumanCyc\", \"x\": -11.39698314666748, \"y\": -13.200993537902832}, {\"pathway\": \"ketolysis\", \"Source\": \"HumanCyc\", \"x\": 13.070555686950684, \"y\": -13.639364242553711}, {\"pathway\": \"glutaryl-CoA degradation\", \"Source\": \"HumanCyc\", \"x\": 12.561283111572266, \"y\": 11.910502433776855}, {\"pathway\": \"mevalonate pathway\", \"Source\": \"HumanCyc\", \"x\": -10.373544692993164, \"y\": -24.32790756225586}, {\"pathway\": \"superpathway of geranylgeranyldiphosphate biosynthesis I (via mevalonate)\", \"Source\": \"HumanCyc\", \"x\": -0.29437750577926636, \"y\": -20.71579360961914}, {\"pathway\": \"trans, trans-farnesyl diphosphate biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -0.023349149152636528, \"y\": 10.597884178161621}, {\"pathway\": \"geranylgeranyldiphosphate biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -29.96918296813965, \"y\": -12.75572395324707}, {\"pathway\": \"L-serine degradation\", \"Source\": \"HumanCyc\", \"x\": -18.301959991455078, \"y\": 1.5553568601608276}, {\"pathway\": \"threonine degradation\", \"Source\": \"HumanCyc\", \"x\": 1.1400147676467896, \"y\": -17.01023292541504}, {\"pathway\": \"glycine cleavage\", \"Source\": \"HumanCyc\", \"x\": -4.161508560180664, \"y\": -12.154993057250977}, {\"pathway\": \"propionyl-CoA degradation\", \"Source\": \"HumanCyc\", \"x\": -14.44518756866455, \"y\": -24.225032806396484}, {\"pathway\": \"2-oxobutanoate degradation\", \"Source\": \"HumanCyc\", \"x\": 17.70798110961914, \"y\": 22.196279525756836}, {\"pathway\": \"superpathway of methionine degradation\", \"Source\": \"HumanCyc\", \"x\": 9.215060234069824, \"y\": -8.209672927856445}, {\"pathway\": \"S-adenosyl-L-methionine biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 11.34789752960205, \"y\": -27.495269775390625}, {\"pathway\": \"methionine degradation\", \"Source\": \"HumanCyc\", \"x\": 14.91201400756836, \"y\": 1.9301940202713013}, {\"pathway\": \"cysteine biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -11.277167320251465, \"y\": 12.21883773803711}, {\"pathway\": \"hydrogen sulfide biosynthesis (trans-sulfuration)\", \"Source\": \"HumanCyc\", \"x\": 15.115377426147461, \"y\": -7.8303542137146}, {\"pathway\": \"cysteine biosynthesis/homocysteine degradation (trans-sulfuration)\", \"Source\": \"HumanCyc\", \"x\": -7.501935005187988, \"y\": -25.30435562133789}, {\"pathway\": \"sulfite oxidation\", \"Source\": \"HumanCyc\", \"x\": 3.7218916416168213, \"y\": -25.106121063232422}, {\"pathway\": \"taurine biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 9.626137733459473, \"y\": 14.379463195800781}, {\"pathway\": \"L-cysteine degradation I\", \"Source\": \"HumanCyc\", \"x\": 0.5028895139694214, \"y\": 22.444551467895508}, {\"pathway\": \"asparagine biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 14.479771614074707, \"y\": 2.9032886028289795}, {\"pathway\": \"aspartate biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -2.106794595718384, \"y\": 3.4877171516418457}, {\"pathway\": \"L-cysteine degradation II\", \"Source\": \"HumanCyc\", \"x\": 7.147192478179932, \"y\": 26.174692153930664}, {\"pathway\": \"asparagine degradation\", \"Source\": \"HumanCyc\", \"x\": -3.182020902633667, \"y\": 11.02599811553955}, {\"pathway\": \"malate-aspartate shuttle\", \"Source\": \"HumanCyc\", \"x\": 10.770818710327148, \"y\": 5.650825023651123}, {\"pathway\": \"4-hydroxyproline degradation\", \"Source\": \"HumanCyc\", \"x\": -27.35462188720703, \"y\": -15.249946594238281}, {\"pathway\": \"proline degradation\", \"Source\": \"HumanCyc\", \"x\": 3.5485317707061768, \"y\": 0.2638947367668152}, {\"pathway\": \"D-mannose degradation\", \"Source\": \"HumanCyc\", \"x\": 12.974143981933594, \"y\": -14.53363037109375}, {\"pathway\": \"GDP-mannose biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -12.456565856933594, \"y\": 25.614032745361328}, {\"pathway\": \"N-acetylglucosamine degradation II\", \"Source\": \"HumanCyc\", \"x\": 13.576556205749512, \"y\": -13.752538681030273}, {\"pathway\": \"N-acetylglucosamine degradation I\", \"Source\": \"HumanCyc\", \"x\": -7.964223861694336, \"y\": -29.8837947845459}, {\"pathway\": \"UDP-N-acetyl-D-galactosamine biosynthesis II\", \"Source\": \"HumanCyc\", \"x\": 6.269328594207764, \"y\": -22.975954055786133}, {\"pathway\": \"UDP-N-acetyl-D-glucosamine biosynthesis II\", \"Source\": \"HumanCyc\", \"x\": 11.376862525939941, \"y\": 17.7571964263916}, {\"pathway\": \"UDP-N-acetyl-D-galactosamine biosynthesis I\", \"Source\": \"HumanCyc\", \"x\": -13.97396469116211, \"y\": 4.287792682647705}, {\"pathway\": \"D-galactose degradation V (Leloir pathway)\", \"Source\": \"HumanCyc\", \"x\": 4.786532878875732, \"y\": 24.593820571899414}, {\"pathway\": \"glycogenolysis\", \"Source\": \"HumanCyc\", \"x\": -22.527265548706055, \"y\": 5.182229042053223}, {\"pathway\": \"GDP-glucose biosynthesis II\", \"Source\": \"HumanCyc\", \"x\": -18.610488891601562, \"y\": -14.581799507141113}, {\"pathway\": \"trehalose degradation\", \"Source\": \"HumanCyc\", \"x\": -16.579553604125977, \"y\": 2.718318223953247}, {\"pathway\": \"NADH repair\", \"Source\": \"HumanCyc\", \"x\": -13.651275634765625, \"y\": 21.118648529052734}, {\"pathway\": \"sucrose degradation\", \"Source\": \"HumanCyc\", \"x\": -15.494221687316895, \"y\": -4.922391414642334}, {\"pathway\": \"glycolysis\", \"Source\": \"HumanCyc\", \"x\": 26.096694946289062, \"y\": -0.045933060348033905}, {\"pathway\": \"gluconeogenesis\", \"Source\": \"HumanCyc\", \"x\": 18.442588806152344, \"y\": 2.8485097885131836}, {\"pathway\": \"Rapoport-Luebering glycolytic shunt\", \"Source\": \"HumanCyc\", \"x\": 6.634280204772949, \"y\": -22.657756805419922}, {\"pathway\": \"D-myo-inositol (1,4,5,6)-tetrakisphosphate biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 5.731514930725098, \"y\": 15.354137420654297}, {\"pathway\": \"D-myo-inositol (3,4,5,6)-tetrakisphosphate biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -27.284366607666016, \"y\": -2.20228910446167}, {\"pathway\": \"1D-myo-inositol hexakisphosphate biosynthesis V (from Ins(1,3,4)P3)\", \"Source\": \"HumanCyc\", \"x\": 10.31196403503418, \"y\": 22.991668701171875}, {\"pathway\": \"inositol pyrophosphates biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 23.621170043945312, \"y\": -9.866904258728027}, {\"pathway\": \"1D-myo-inositol hexakisphosphate biosynthesis II (mammalian)\", \"Source\": \"HumanCyc\", \"x\": 24.18222427368164, \"y\": -13.43264102935791}, {\"pathway\": \"D-myo-inositol (1,3,4)-trisphosphate biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 22.34296989440918, \"y\": 7.4631500244140625}, {\"pathway\": \"superpathway of D-myo-inositol (1,4,5)-trisphosphate metabolism\", \"Source\": \"HumanCyc\", \"x\": 9.504568099975586, \"y\": 26.559852600097656}, {\"pathway\": \"D-myo-inositol (1,4,5)-trisphosphate degradation\", \"Source\": \"HumanCyc\", \"x\": 14.718084335327148, \"y\": -24.670989990234375}, {\"pathway\": \"3-phosphoinositide degradation\", \"Source\": \"HumanCyc\", \"x\": 3.694108724594116, \"y\": 20.298877716064453}, {\"pathway\": \"superpathway of inositol phosphate compounds\", \"Source\": \"HumanCyc\", \"x\": -9.625442504882812, \"y\": -1.9169867038726807}, {\"pathway\": \"3-phosphoinositide biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 14.885441780090332, \"y\": -14.852518081665039}, {\"pathway\": \"D-myo-inositol (1,4,5)-trisphosphate biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 3.3279571533203125, \"y\": -1.489918828010559}, {\"pathway\": \"D-myo-inositol-5-phosphate metabolism\", \"Source\": \"HumanCyc\", \"x\": -23.450870513916016, \"y\": -7.141242027282715}, {\"pathway\": \"phospholipases\", \"Source\": \"HumanCyc\", \"x\": -0.6436197757720947, \"y\": 19.150604248046875}, {\"pathway\": \"myo-inositol de novo biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -11.855437278747559, \"y\": -21.465669631958008}, {\"pathway\": \"acyl-CoA hydrolysis\", \"Source\": \"HumanCyc\", \"x\": 19.776010513305664, \"y\": -6.662519931793213}, {\"pathway\": \"stearate biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -4.415592670440674, \"y\": 3.0721359252929688}, {\"pathway\": \"fatty acid beta-oxidation\", \"Source\": \"HumanCyc\", \"x\": 13.6227445602417, \"y\": -27.82943344116211}, {\"pathway\": \"fatty acid beta-oxidation (peroxisome)\", \"Source\": \"HumanCyc\", \"x\": 21.923099517822266, \"y\": -4.207080364227295}, {\"pathway\": \"fatty acid activation\", \"Source\": \"HumanCyc\", \"x\": 0.9546568393707275, \"y\": -3.041428565979004}, {\"pathway\": \"gamma-linolenate biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -26.993587493896484, \"y\": -8.228424072265625}, {\"pathway\": \"eicosapentaenoate biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -27.80169105529785, \"y\": 3.986560583114624}, {\"pathway\": \"oleate biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -9.421475410461426, \"y\": 27.65092658996582}, {\"pathway\": \"bile acid biosynthesis, neutral pathway\", \"Source\": \"HumanCyc\", \"x\": 7.672915935516357, \"y\": 10.837774276733398}, {\"pathway\": \"1,25-dihydroxyvitamin D3 biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -13.411111831665039, \"y\": -26.91487693786621}, {\"pathway\": \"fatty acid beta-oxidation (unsaturated, odd number)\", \"Source\": \"HumanCyc\", \"x\": 14.472894668579102, \"y\": 25.594636917114258}, {\"pathway\": \"thyroid hormone metabolism I (via deiodination)\", \"Source\": \"HumanCyc\", \"x\": 8.530313491821289, \"y\": 25.939664840698242}, {\"pathway\": \"thyronamine and iodothyronamine metabolism\", \"Source\": \"HumanCyc\", \"x\": 10.914886474609375, \"y\": -19.693620681762695}, {\"pathway\": \"thyroid hormone metabolism II (via conjugation and/or degradation)\", \"Source\": \"HumanCyc\", \"x\": -12.79371452331543, \"y\": 3.8741729259490967}, {\"pathway\": \"superoxide radicals degradation\", \"Source\": \"HumanCyc\", \"x\": -5.2813286781311035, \"y\": -22.676118850708008}, {\"pathway\": \"methylglyoxal degradation III\", \"Source\": \"HumanCyc\", \"x\": -10.502114295959473, \"y\": 20.593334197998047}, {\"pathway\": \"spermine and spermidine degradation I\", \"Source\": \"HumanCyc\", \"x\": -3.685506582260132, \"y\": -28.364715576171875}, {\"pathway\": \"serotonin and melatonin biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 20.405637741088867, \"y\": 22.573265075683594}, {\"pathway\": \"catecholamine biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -23.08002471923828, \"y\": 4.49493408203125}, {\"pathway\": \"tryptophan degradation X (mammalian, via tryptamine)\", \"Source\": \"HumanCyc\", \"x\": 26.691970825195312, \"y\": -3.6760900020599365}, {\"pathway\": \"fatty acid alpha-oxidation\", \"Source\": \"HumanCyc\", \"x\": 0.48460450768470764, \"y\": -16.523794174194336}, {\"pathway\": \"fatty acid alpha-oxidation III\", \"Source\": \"HumanCyc\", \"x\": -24.874387741088867, \"y\": -8.136178970336914}, {\"pathway\": \"phytol degradation\", \"Source\": \"HumanCyc\", \"x\": 4.371439456939697, \"y\": 10.846232414245605}, {\"pathway\": \"phenylethylamine degradation I\", \"Source\": \"HumanCyc\", \"x\": 13.007647514343262, \"y\": -0.31347474455833435}, {\"pathway\": \"ethanol degradation II\", \"Source\": \"HumanCyc\", \"x\": -26.22632598876953, \"y\": -7.205804824829102}, {\"pathway\": \"oxidative ethanol degradation III\", \"Source\": \"HumanCyc\", \"x\": 25.94872283935547, \"y\": 3.0868189334869385}, {\"pathway\": \"ethanol degradation IV\", \"Source\": \"HumanCyc\", \"x\": 5.722014904022217, \"y\": 17.623367309570312}, {\"pathway\": \"acetate conversion to acetyl-CoA\", \"Source\": \"HumanCyc\", \"x\": 5.155632972717285, \"y\": 26.693193435668945}, {\"pathway\": \"L-dopa degradation\", \"Source\": \"HumanCyc\", \"x\": -21.501670837402344, \"y\": -3.809349298477173}, {\"pathway\": \"dopamine degradation\", \"Source\": \"HumanCyc\", \"x\": -3.249178886413574, \"y\": 19.63970375061035}, {\"pathway\": \"noradrenaline and adrenaline degradation\", \"Source\": \"HumanCyc\", \"x\": -17.642162322998047, \"y\": 2.230999231338501}, {\"pathway\": \"serotonin degradation\", \"Source\": \"HumanCyc\", \"x\": -23.7800350189209, \"y\": -6.89565896987915}, {\"pathway\": \"melatonin degradation II\", \"Source\": \"HumanCyc\", \"x\": 0.014093291014432907, \"y\": 1.8095130920410156}, {\"pathway\": \"putrescine degradation III\", \"Source\": \"HumanCyc\", \"x\": 10.297260284423828, \"y\": 4.652881622314453}, {\"pathway\": \"2-deoxy-alpha-D-ribose 1-phosphate degradation\", \"Source\": \"HumanCyc\", \"x\": -3.8299334049224854, \"y\": -6.346106052398682}, {\"pathway\": \"purine ribonucleosides degradation to ribose-1-phosphate\", \"Source\": \"HumanCyc\", \"x\": 1.8464773893356323, \"y\": -7.564165115356445}, {\"pathway\": \"purine deoxyribonucleosides degradation\", \"Source\": \"HumanCyc\", \"x\": -9.810023307800293, \"y\": -4.434299945831299}, {\"pathway\": \"adenine and adenosine salvage III\", \"Source\": \"HumanCyc\", \"x\": 7.937709331512451, \"y\": -14.967079162597656}, {\"pathway\": \"guanine and guanosine salvage\", \"Source\": \"HumanCyc\", \"x\": 17.4256534576416, \"y\": 2.141831159591675}, {\"pathway\": \"guanosine nucleotides degradation\", \"Source\": \"HumanCyc\", \"x\": 14.909433364868164, \"y\": 8.898161888122559}, {\"pathway\": \"urate biosynthesis/inosine 5-phosphate degradation\", \"Source\": \"HumanCyc\", \"x\": -2.6755528450012207, \"y\": -14.003931999206543}, {\"pathway\": \"purine nucleotides degradation\", \"Source\": \"HumanCyc\", \"x\": -26.45598602294922, \"y\": -15.77371597290039}, {\"pathway\": \"adenosine nucleotides degradation\", \"Source\": \"HumanCyc\", \"x\": 8.33206844329834, \"y\": 7.889681816101074}, {\"pathway\": \"retinoate biosynthesis II\", \"Source\": \"HumanCyc\", \"x\": 4.8810553550720215, \"y\": 8.235755920410156}, {\"pathway\": \"retinoate biosynthesis I\", \"Source\": \"HumanCyc\", \"x\": 12.682469367980957, \"y\": 0.47341975569725037}, {\"pathway\": \"the visual cycle I (vertebrates)\", \"Source\": \"HumanCyc\", \"x\": -2.3181211948394775, \"y\": -11.521557807922363}, {\"pathway\": \"retinol biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 4.030986309051514, \"y\": 4.625053882598877}, {\"pathway\": \"triacylglycerol degradation\", \"Source\": \"HumanCyc\", \"x\": 23.802974700927734, \"y\": -12.509710311889648}, {\"pathway\": \"thymine degradation\", \"Source\": \"HumanCyc\", \"x\": -0.923118531703949, \"y\": 10.96359920501709}, {\"pathway\": \"uracil degradation\", \"Source\": \"HumanCyc\", \"x\": -24.608896255493164, \"y\": -10.538521766662598}, {\"pathway\": \"pyrimidine ribonucleosides degradation\", \"Source\": \"HumanCyc\", \"x\": 22.799453735351562, \"y\": -13.827766418457031}, {\"pathway\": \"pyrimidine deoxyribonucleosides degradation\", \"Source\": \"HumanCyc\", \"x\": -15.022978782653809, \"y\": -26.420656204223633}, {\"pathway\": \"pyrimidine ribonucleosides salvage I\", \"Source\": \"HumanCyc\", \"x\": 2.7651124000549316, \"y\": -5.739201068878174}, {\"pathway\": \"pyrimidine deoxyribonucleosides salvage\", \"Source\": \"HumanCyc\", \"x\": -25.5283260345459, \"y\": 6.503694534301758}, {\"pathway\": \"superpathway of pyrimidine deoxyribonucleoside salvage\", \"Source\": \"HumanCyc\", \"x\": -18.60382652282715, \"y\": -16.48788833618164}, {\"pathway\": \"pyrimidine deoxyribonucleotide phosphorylation\", \"Source\": \"HumanCyc\", \"x\": -5.0079169273376465, \"y\": 16.29942512512207}, {\"pathway\": \"CMP phosphorylation\", \"Source\": \"HumanCyc\", \"x\": -7.056422233581543, \"y\": -27.225217819213867}, {\"pathway\": \"UTP and CTP de novo biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 24.771364212036133, \"y\": 15.845428466796875}, {\"pathway\": \"superpathway of pyrimidine ribonucleotides de novo biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -7.1004509925842285, \"y\": 16.165355682373047}, {\"pathway\": \"superpathway of pyrimidine deoxyribonucleotides de novo biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -14.081144332885742, \"y\": -11.345012664794922}, {\"pathway\": \"pyrimidine deoxyribonucleotides de novo biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -12.878141403198242, \"y\": 28.211700439453125}, {\"pathway\": \"pyrimidine deoxyribonucleotides biosynthesis from CTP\", \"Source\": \"HumanCyc\", \"x\": -9.655346870422363, \"y\": -10.574268341064453}, {\"pathway\": \"guanosine deoxyribonucleotides de novo biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -12.327341079711914, \"y\": 5.819796085357666}, {\"pathway\": \"adenosine deoxyribonucleotides de novo biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 12.144405364990234, \"y\": -14.744449615478516}, {\"pathway\": \"guanosine nucleotides de novo biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 7.427101135253906, \"y\": -25.0924072265625}, {\"pathway\": \"guanosine ribonucleotides de novo biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -17.935321807861328, \"y\": -24.804664611816406}, {\"pathway\": \"purine deoxyribonucleosides salvage\", \"Source\": \"HumanCyc\", \"x\": 6.510496616363525, \"y\": 11.329703330993652}, {\"pathway\": \"purine nucleotides de novo biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 10.094645500183105, \"y\": -15.682578086853027}, {\"pathway\": \"superpathway of purine nucleotide salvage\", \"Source\": \"HumanCyc\", \"x\": -9.239750862121582, \"y\": 26.687891006469727}, {\"pathway\": \"adenosine ribonucleotides de novo biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -2.721536159515381, \"y\": 29.234933853149414}, {\"pathway\": \"UTP and CTP dephosphorylation II\", \"Source\": \"HumanCyc\", \"x\": -1.705726146697998, \"y\": -26.4993953704834}, {\"pathway\": \"UTP and CTP dephosphorylation I\", \"Source\": \"HumanCyc\", \"x\": 1.2135165929794312, \"y\": -15.982552528381348}, {\"pathway\": \"UMP biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 22.635791778564453, \"y\": -3.3052706718444824}, {\"pathway\": \"inosine-5-phosphate biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 15.839815139770508, \"y\": 7.993771553039551}, {\"pathway\": \"adenine and adenosine salvage II\", \"Source\": \"HumanCyc\", \"x\": -3.1693673133850098, \"y\": -11.977145195007324}, {\"pathway\": \"adenine and adenosine salvage I\", \"Source\": \"HumanCyc\", \"x\": 15.975436210632324, \"y\": 10.988269805908203}, {\"pathway\": \"NAD salvage\", \"Source\": \"HumanCyc\", \"x\": -1.790033221244812, \"y\": -12.669361114501953}, {\"pathway\": \"NAD biosynthesis from 2-amino-3-carboxymuconate semialdehyde\", \"Source\": \"HumanCyc\", \"x\": -4.868905544281006, \"y\": -4.2893853187561035}, {\"pathway\": \"NAD de novo biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -1.68081796169281, \"y\": 20.26251983642578}, {\"pathway\": \"tryptophan degradation to 2-amino-3-carboxymuconate semialdehyde\", \"Source\": \"HumanCyc\", \"x\": 6.741386890411377, \"y\": 8.17226505279541}, {\"pathway\": \"tryptophan degradation\", \"Source\": \"HumanCyc\", \"x\": 4.157884120941162, \"y\": 5.094918727874756}, {\"pathway\": \"L-kynurenine degradation\", \"Source\": \"HumanCyc\", \"x\": 1.9407094717025757, \"y\": -2.8966526985168457}, {\"pathway\": \"2-amino-3-carboxymuconate semialdehyde degradation to glutaryl-CoA\", \"Source\": \"HumanCyc\", \"x\": 21.90558624267578, \"y\": 17.335115432739258}, {\"pathway\": \"lysine degradation II (pipecolate pathway)\", \"Source\": \"HumanCyc\", \"x\": -8.767107963562012, \"y\": -1.2899894714355469}, {\"pathway\": \"lysine degradation I (saccharopine pathway)\", \"Source\": \"HumanCyc\", \"x\": -3.6902272701263428, \"y\": 5.206630229949951}, {\"pathway\": \"choline degradation\", \"Source\": \"HumanCyc\", \"x\": 2.54046368598938, \"y\": 15.254326820373535}, {\"pathway\": \"superpathway of choline degradation to L-serine\", \"Source\": \"HumanCyc\", \"x\": 0.8509748578071594, \"y\": 4.871570110321045}, {\"pathway\": \"glycine betaine degradation\", \"Source\": \"HumanCyc\", \"x\": 10.647340774536133, \"y\": -12.012211799621582}, {\"pathway\": \"glycine/serine biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -14.6687593460083, \"y\": -10.910762786865234}, {\"pathway\": \"folate polyglutamylation\", \"Source\": \"HumanCyc\", \"x\": -24.10196876525879, \"y\": 8.651032447814941}, {\"pathway\": \"folate transformations\", \"Source\": \"HumanCyc\", \"x\": 19.645225524902344, \"y\": 9.9629487991333}, {\"pathway\": \"dTMP de novo biosynthesis (mitochondrial)\", \"Source\": \"HumanCyc\", \"x\": -12.807770729064941, \"y\": -0.9151050448417664}, {\"pathway\": \"serine and glycine biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 1.1021286249160767, \"y\": 0.9175342917442322}, {\"pathway\": \"serine biosynthesis (phosphorylated route)\", \"Source\": \"HumanCyc\", \"x\": -9.189937591552734, \"y\": -12.745481491088867}, {\"pathway\": \"methionine salvage\", \"Source\": \"HumanCyc\", \"x\": 13.217578887939453, \"y\": 22.97259521484375}, {\"pathway\": \"histidine degradation\", \"Source\": \"HumanCyc\", \"x\": 23.107717514038086, \"y\": 16.718612670898438}, {\"pathway\": \"tetrahydrofolate salvage from 5,10-methenyltetrahydrofolate\", \"Source\": \"HumanCyc\", \"x\": -14.434365272521973, \"y\": -29.193683624267578}, {\"pathway\": \"5-aminoimidazole ribonucleotide biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -16.706483840942383, \"y\": 18.413667678833008}, {\"pathway\": \"proline biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 8.1860933303833, \"y\": 20.51399803161621}, {\"pathway\": \"ornithine de novo biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 12.768684387207031, \"y\": 24.749048233032227}, {\"pathway\": \"glutamate biosynthesis/degradation\", \"Source\": \"HumanCyc\", \"x\": -5.516103267669678, \"y\": -22.346965789794922}, {\"pathway\": \"GABA shunt\", \"Source\": \"HumanCyc\", \"x\": -12.53908634185791, \"y\": 10.895599365234375}, {\"pathway\": \"glutamate dependent acid resistance\", \"Source\": \"HumanCyc\", \"x\": -12.62686538696289, \"y\": -14.963086128234863}, {\"pathway\": \"glutamine biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 20.267528533935547, \"y\": -9.650561332702637}, {\"pathway\": \"4-aminobutyrate degradation\", \"Source\": \"HumanCyc\", \"x\": 7.547641754150391, \"y\": 26.701576232910156}, {\"pathway\": \"beta-alanine degradation\", \"Source\": \"HumanCyc\", \"x\": 10.891231536865234, \"y\": 15.276643753051758}, {\"pathway\": \"protein O-[N-acetyl]-glucosylation\", \"Source\": \"HumanCyc\", \"x\": 14.535229682922363, \"y\": -27.429908752441406}, {\"pathway\": \"protein citrullination\", \"Source\": \"HumanCyc\", \"x\": -14.77476978302002, \"y\": -29.932125091552734}, {\"pathway\": \"creatine biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 0.3155059814453125, \"y\": 17.917545318603516}, {\"pathway\": \"BMP Signalling Pathway\", \"Source\": \"HumanCyc\", \"x\": -1.3603862524032593, \"y\": -6.011806964874268}, {\"pathway\": \"coenzyme A biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 2.781078577041626, \"y\": -6.900156021118164}, {\"pathway\": \"CMP-N-acetylneuraminate biosynthesis I (eukaryotes)\", \"Source\": \"HumanCyc\", \"x\": 2.0662102699279785, \"y\": -5.9325971603393555}, {\"pathway\": \"putrescine biosynthesis I\", \"Source\": \"HumanCyc\", \"x\": -4.676632881164551, \"y\": -1.9230679273605347}, {\"pathway\": \"putrescine biosynthesis II\", \"Source\": \"HumanCyc\", \"x\": 11.985757827758789, \"y\": 9.171813011169434}, {\"pathway\": \"PRPP biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 3.3252065181732178, \"y\": 26.799760818481445}, {\"pathway\": \"pyridoxal 5-phosphate salvage\", \"Source\": \"HumanCyc\", \"x\": 12.353540420532227, \"y\": -21.532323837280273}, {\"pathway\": \"lactose degradation III\", \"Source\": \"HumanCyc\", \"x\": 17.08498191833496, \"y\": 17.952085494995117}, {\"pathway\": \"diphthamide biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -5.888890743255615, \"y\": -9.241276741027832}, {\"pathway\": \"dermatan sulfate biosynthesis (late stages)\", \"Source\": \"HumanCyc\", \"x\": 20.118806838989258, \"y\": 2.439880609512329}, {\"pathway\": \"mRNA capping\", \"Source\": \"HumanCyc\", \"x\": 22.382915496826172, \"y\": 18.167661666870117}, {\"pathway\": \"progesterone biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -8.229693412780762, \"y\": 12.480833053588867}, {\"pathway\": \"androgen biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 10.538544654846191, \"y\": 4.481594562530518}, {\"pathway\": \"allopregnanolone biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 21.843368530273438, \"y\": 8.845888137817383}, {\"pathway\": \"creatine-phosphate biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 13.870280265808105, \"y\": 14.41685676574707}, {\"pathway\": \"MAP kinase cascade\", \"Source\": \"HumanCyc\", \"x\": 10.683784484863281, \"y\": 20.74602699279785}, {\"pathway\": \"fructose 2,6-bisphosphate synthesis/dephosphorylation\", \"Source\": \"HumanCyc\", \"x\": -14.030014038085938, \"y\": -0.9941334128379822}, {\"pathway\": \"histamine degradation\", \"Source\": \"HumanCyc\", \"x\": 21.697296142578125, \"y\": 0.25617292523384094}, {\"pathway\": \"S-methyl-5-thioadenosine degradation\", \"Source\": \"HumanCyc\", \"x\": 15.406844139099121, \"y\": 2.3223989009857178}, {\"pathway\": \"methylthiopropionate biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 18.948421478271484, \"y\": 2.302525520324707}, {\"pathway\": \"S-methyl-5-thio-alpha-D-ribose 1-phosphate degradation\", \"Source\": \"HumanCyc\", \"x\": -12.907977104187012, \"y\": 13.242612838745117}, {\"pathway\": \"sorbitol degradation I\", \"Source\": \"HumanCyc\", \"x\": -14.028385162353516, \"y\": 8.976154327392578}, {\"pathway\": \"spermidine biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -27.692380905151367, \"y\": 3.1642894744873047}, {\"pathway\": \"UDP-D-xylose and UDP-D-glucuronate biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -1.079540491104126, \"y\": -19.92379379272461}, {\"pathway\": \"spermine biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -17.856409072875977, \"y\": 0.4110950529575348}, {\"pathway\": \"sphingomyelin metabolism/ceramide salvage\", \"Source\": \"HumanCyc\", \"x\": 7.7721686363220215, \"y\": 11.683213233947754}, {\"pathway\": \"sphingosine and sphingosine-1-phosphate metabolism\", \"Source\": \"HumanCyc\", \"x\": -4.393144607543945, \"y\": -25.549121856689453}, {\"pathway\": \"L-carnitine biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -6.314794063568115, \"y\": 26.376554489135742}, {\"pathway\": \"sulfate activation for sulfonation\", \"Source\": \"HumanCyc\", \"x\": 10.282368659973145, \"y\": -15.546917915344238}, {\"pathway\": \"dolichyl-diphosphooligosaccharide biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 12.624939918518066, \"y\": -9.593876838684082}, {\"pathway\": \"histamine biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 27.75737762451172, \"y\": -6.416062355041504}, {\"pathway\": \"glutamine degradation/glutamate biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -12.279729843139648, \"y\": -14.319021224975586}, {\"pathway\": \"alanine biosynthesis/degradation\", \"Source\": \"HumanCyc\", \"x\": -23.118194580078125, \"y\": 1.3213270902633667}, {\"pathway\": \"molybdenum cofactor biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 6.908255577087402, \"y\": -26.110761642456055}, {\"pathway\": \"iron-sulfur cluster biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -18.548667907714844, \"y\": -1.5161815881729126}, {\"pathway\": \"heme degradation\", \"Source\": \"HumanCyc\", \"x\": -25.144290924072266, \"y\": 9.772395133972168}, {\"pathway\": \"ceramide de novo biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 1.9861265420913696, \"y\": 23.84952735900879}, {\"pathway\": \"eumelanin biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 20.328445434570312, \"y\": 10.958586692810059}, {\"pathway\": \"L-dopachrome biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -10.25694465637207, \"y\": 3.983346462249756}, {\"pathway\": \"(S)-reticuline biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -23.584238052368164, \"y\": -1.7740479707717896}, {\"pathway\": \"dolichol and dolichyl phosphate biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -24.940204620361328, \"y\": -1.683297038078308}, {\"pathway\": \"GDP-L-fucose biosynthesis I (from GDP-D-mannose)\", \"Source\": \"HumanCyc\", \"x\": 13.604393005371094, \"y\": 1.1925666332244873}, {\"pathway\": \"tRNA charging\", \"Source\": \"HumanCyc\", \"x\": 6.983633995056152, \"y\": 27.72357749938965}, {\"pathway\": \"selenocysteine biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -1.7183762788772583, \"y\": 27.932247161865234}, {\"pathway\": \"tRNA splicing\", \"Source\": \"HumanCyc\", \"x\": -16.97281837463379, \"y\": -0.667070209980011}, {\"pathway\": \"terminal O-glycans residues modification\", \"Source\": \"HumanCyc\", \"x\": -9.693354606628418, \"y\": -28.437894821166992}, {\"pathway\": \"mucin core 1 and core 2 O-glycosylation\", \"Source\": \"HumanCyc\", \"x\": 14.426974296569824, \"y\": -1.9529495239257812}, {\"pathway\": \"tetrahydrobiopterin de novo biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 21.945375442504883, \"y\": -9.444464683532715}, {\"pathway\": \"cardiolipin biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 26.375490188598633, \"y\": -0.5551238059997559}, {\"pathway\": \"tetrapyrrole biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -1.9485174417495728, \"y\": -6.424105167388916}, {\"pathway\": \"heme biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 12.09468936920166, \"y\": 15.105621337890625}, {\"pathway\": \"heme biosynthesis from uroporphyrinogen-III I\", \"Source\": \"HumanCyc\", \"x\": 19.17426872253418, \"y\": 15.722814559936523}, {\"pathway\": \"NAD phosphorylation and dephosphorylation\", \"Source\": \"HumanCyc\", \"x\": 3.470052480697632, \"y\": 16.374502182006836}, {\"pathway\": \"thiamin salvage III\", \"Source\": \"HumanCyc\", \"x\": -9.971641540527344, \"y\": -1.6801457405090332}, {\"pathway\": \"thio-molybdenum cofactor biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 7.362978458404541, \"y\": -21.889822006225586}, {\"pathway\": \"thioredoxin pathway\", \"Source\": \"HumanCyc\", \"x\": -10.88491439819336, \"y\": -22.317920684814453}, {\"pathway\": \"thiosulfate disproportionation III (rhodanese)\", \"Source\": \"HumanCyc\", \"x\": -10.168195724487305, \"y\": 6.01906681060791}, {\"pathway\": \"formaldehyde oxidation\", \"Source\": \"HumanCyc\", \"x\": -3.2215232849121094, \"y\": 0.859510064125061}, {\"pathway\": \"thyroid hormone biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -6.716049671173096, \"y\": 27.06525993347168}, {\"pathway\": \"glutathione biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 17.361328125, \"y\": 22.3983154296875}, {\"pathway\": \"gamma-glutamyl cycle\", \"Source\": \"HumanCyc\", \"x\": -20.23988914489746, \"y\": 9.62099552154541}, {\"pathway\": \"leukotriene biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -24.457876205444336, \"y\": 9.450833320617676}, {\"pathway\": \"lipoxin biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -6.216145992279053, \"y\": -11.930885314941406}, {\"pathway\": \"resolvin D biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -7.249298572540283, \"y\": 1.00811767578125}, {\"pathway\": \"aspirin triggered resolvin E biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -17.716054916381836, \"y\": 18.358531951904297}, {\"pathway\": \"aspirin-triggered lipoxin biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -7.371852397918701, \"y\": 13.57505989074707}, {\"pathway\": \"aspirin triggered resolvin D biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 16.363128662109375, \"y\": 23.745893478393555}, {\"pathway\": \"C20 prostanoid biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 23.641756057739258, \"y\": 7.832516193389893}, {\"pathway\": \"dermatan sulfate degradation (metazoa)\", \"Source\": \"HumanCyc\", \"x\": 11.915403366088867, \"y\": -4.025627136230469}, {\"pathway\": \"chondroitin sulfate degradation (metazoa)\", \"Source\": \"HumanCyc\", \"x\": 7.758837699890137, \"y\": -27.853574752807617}, {\"pathway\": \"acyl carrier protein metabolism\", \"Source\": \"HumanCyc\", \"x\": 5.025696754455566, \"y\": -2.7143242359161377}, {\"pathway\": \"fatty acid biosynthesis initiation\", \"Source\": \"HumanCyc\", \"x\": -1.199054479598999, \"y\": -5.207785129547119}, {\"pathway\": \"acetyl-CoA biosynthesis from citrate\", \"Source\": \"HumanCyc\", \"x\": 11.358352661132812, \"y\": -20.519575119018555}, {\"pathway\": \"triacylglycerol biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 9.278707504272461, \"y\": -21.622831344604492}, {\"pathway\": \"CDP-diacylglycerol biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -4.833586692810059, \"y\": 6.252027988433838}, {\"pathway\": \"glycerol degradation\", \"Source\": \"HumanCyc\", \"x\": 4.561288356781006, \"y\": -11.99560260772705}, {\"pathway\": \"glycerol-3-phosphate shuttle\", \"Source\": \"HumanCyc\", \"x\": -6.986082553863525, \"y\": -14.417524337768555}, {\"pathway\": \"GDP-L-fucose biosynthesis II (from L-fucose)\", \"Source\": \"HumanCyc\", \"x\": 8.43134593963623, \"y\": -1.1435831785202026}, {\"pathway\": \"ascorbate recycling (cytosolic)\", \"Source\": \"HumanCyc\", \"x\": -5.142292499542236, \"y\": 7.437652111053467}, {\"pathway\": \"homocarnosine biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -21.611753463745117, \"y\": 10.821258544921875}, {\"pathway\": \"carnosine biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 11.478848457336426, \"y\": 18.965295791625977}, {\"pathway\": \"4-hydroxybenzoate biosynthesis\", \"Source\": \"HumanCyc\", \"x\": 25.68425941467285, \"y\": -7.520827770233154}, {\"pathway\": \"tyrosine degradation\", \"Source\": \"HumanCyc\", \"x\": 0.011068950407207012, \"y\": -5.725165843963623}, {\"pathway\": \"glutamate removal from folates\", \"Source\": \"HumanCyc\", \"x\": 16.638294219970703, \"y\": 9.09712028503418}, {\"pathway\": \"glutathione-mediated detoxification\", \"Source\": \"HumanCyc\", \"x\": -16.96366310119629, \"y\": -24.08379554748535}, {\"pathway\": \"4-hydroxy-2-nonenal detoxification\", \"Source\": \"HumanCyc\", \"x\": 12.516308784484863, \"y\": 5.202597618103027}, {\"pathway\": \"ubiquinol-10 biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -17.883756637573242, \"y\": -24.25980567932129}, {\"pathway\": \"glutathione redox reactions I\", \"Source\": \"HumanCyc\", \"x\": 11.158940315246582, \"y\": -12.2797269821167}, {\"pathway\": \"glycine biosynthesis\", \"Source\": \"HumanCyc\", \"x\": -3.377087116241455, \"y\": 29.122682571411133}, {\"pathway\": \"pathogenic escherichia coli infection ehec\", \"Source\": \"Kegg\", \"x\": 6.456647872924805, \"y\": 26.559179306030273}, {\"pathway\": \"pathogenic escherichia coli infection epec\", \"Source\": \"Kegg\", \"x\": -28.91007423400879, \"y\": -6.501236438751221}, {\"pathway\": \"adherens junction\", \"Source\": \"Kegg\", \"x\": -5.629944324493408, \"y\": 6.521791934967041}, {\"pathway\": \"leukocyte transendothelial migration\", \"Source\": \"Kegg\", \"x\": -8.942719459533691, \"y\": 5.632516860961914}, {\"pathway\": \"tight junction\", \"Source\": \"Kegg\", \"x\": -15.327569007873535, \"y\": -10.393729209899902}, {\"pathway\": \"axon guidance\", \"Source\": \"Kegg\", \"x\": -12.361359596252441, \"y\": -16.074861526489258}, {\"pathway\": \"acute myeloid leukemia\", \"Source\": \"Kegg\", \"x\": -10.745234489440918, \"y\": -7.13264799118042}, {\"pathway\": \"chronic myeloid leukemia\", \"Source\": \"Kegg\", \"x\": 7.5973663330078125, \"y\": -0.28670260310173035}, {\"pathway\": \"pancreatic cancer\", \"Source\": \"Kegg\", \"x\": 23.84893226623535, \"y\": 17.006338119506836}, {\"pathway\": \"endometrial cancer\", \"Source\": \"Kegg\", \"x\": 8.14240837097168, \"y\": -7.433220386505127}, {\"pathway\": \"prostate cancer\", \"Source\": \"Kegg\", \"x\": -13.021498680114746, \"y\": 11.077081680297852}, {\"pathway\": \"glioma\", \"Source\": \"Kegg\", \"x\": 6.775938034057617, \"y\": 1.3008009195327759}, {\"pathway\": \"non small cell lung cancer\", \"Source\": \"Kegg\", \"x\": 26.750123977661133, \"y\": -9.903782844543457}, {\"pathway\": \"melanoma\", \"Source\": \"Kegg\", \"x\": 26.594266891479492, \"y\": -7.830187797546387}, {\"pathway\": \"colorectal cancer\", \"Source\": \"Kegg\", \"x\": 0.12578102946281433, \"y\": -7.227941036224365}, {\"pathway\": \"erbb signaling pathway\", \"Source\": \"Kegg\", \"x\": 30.414522171020508, \"y\": -8.135713577270508}, {\"pathway\": \"renal cell carcinoma\", \"Source\": \"Kegg\", \"x\": 15.108382225036621, \"y\": -25.399553298950195}, {\"pathway\": \"bladder cancer\", \"Source\": \"Kegg\", \"x\": -15.317658424377441, \"y\": -0.6439387798309326}, {\"pathway\": \"thyroid cancer\", \"Source\": \"Kegg\", \"x\": -19.931194305419922, \"y\": -15.08572769165039}, {\"pathway\": \"b cell receptor signaling pathway\", \"Source\": \"Kegg\", \"x\": -3.3333041667938232, \"y\": -10.357771873474121}, {\"pathway\": \"t cell receptor signaling pathway\", \"Source\": \"Kegg\", \"x\": -11.359681129455566, \"y\": -7.461202621459961}, {\"pathway\": \"fc epsilon ri signaling pathway\", \"Source\": \"Kegg\", \"x\": 20.995534896850586, \"y\": -9.000554084777832}, {\"pathway\": \"vegf signaling pathway\", \"Source\": \"Kegg\", \"x\": -22.29511833190918, \"y\": 6.206057548522949}, {\"pathway\": \"natural killer cell mediated cytotoxicity\", \"Source\": \"Kegg\", \"x\": -24.933181762695312, \"y\": -13.026257514953613}, {\"pathway\": \"toll like receptor signaling pathway\", \"Source\": \"Kegg\", \"x\": 15.685928344726562, \"y\": 12.217745780944824}, {\"pathway\": \"insulin signaling pathway\", \"Source\": \"Kegg\", \"x\": -3.5539965629577637, \"y\": -3.301654577255249}, {\"pathway\": \"type ii diabetes mellitus\", \"Source\": \"Kegg\", \"x\": 20.40747833251953, \"y\": -7.526569843292236}, {\"pathway\": \"mtor signaling pathway\", \"Source\": \"Kegg\", \"x\": -7.910799503326416, \"y\": -12.480338096618652}, {\"pathway\": \"apoptosis\", \"Source\": \"Kegg\", \"x\": -14.594837188720703, \"y\": -14.824645042419434}, {\"pathway\": \"small cell lung cancer\", \"Source\": \"Kegg\", \"x\": -20.793651580810547, \"y\": -6.512550354003906}, {\"pathway\": \"mapk signaling pathway\", \"Source\": \"Kegg\", \"x\": -7.070256233215332, \"y\": 28.169462203979492}, {\"pathway\": \"regulation of actin cytoskeleton\", \"Source\": \"Kegg\", \"x\": -8.753979682922363, \"y\": -16.72780418395996}, {\"pathway\": \"dorso ventral axis formation\", \"Source\": \"Kegg\", \"x\": -12.922525405883789, \"y\": 26.161983489990234}, {\"pathway\": \"inositol phosphate metabolism\", \"Source\": \"Kegg\", \"x\": -20.964479446411133, \"y\": -12.06865119934082}, {\"pathway\": \"phosphatidylinositol signaling system\", \"Source\": \"Kegg\", \"x\": 18.132747650146484, \"y\": 16.997987747192383}, {\"pathway\": \"cell communication\", \"Source\": \"Kegg\", \"x\": -10.81892204284668, \"y\": 26.29244613647461}, {\"pathway\": \"ecm receptor interaction\", \"Source\": \"Kegg\", \"x\": 27.56747055053711, \"y\": -11.171076774597168}, {\"pathway\": \"focal adhesion\", \"Source\": \"Kegg\", \"x\": -22.510967254638672, \"y\": 6.988556861877441}, {\"pathway\": \"basal cell carcinoma\", \"Source\": \"Kegg\", \"x\": -3.493284225463867, \"y\": 12.466570854187012}, {\"pathway\": \"hedgehog signaling pathway\", \"Source\": \"Kegg\", \"x\": -2.2325916290283203, \"y\": 16.023630142211914}, {\"pathway\": \"melanogenesis\", \"Source\": \"Kegg\", \"x\": -23.763137817382812, \"y\": 2.1912503242492676}, {\"pathway\": \"wnt signaling pathway\", \"Source\": \"Kegg\", \"x\": 18.707672119140625, \"y\": 9.95046329498291}, {\"pathway\": \"calcium signaling pathway\", \"Source\": \"Kegg\", \"x\": 11.206999778747559, \"y\": 6.319146156311035}, {\"pathway\": \"neuroactive ligand receptor interaction\", \"Source\": \"Kegg\", \"x\": -0.38086575269699097, \"y\": -14.597029685974121}, {\"pathway\": \"gap junction\", \"Source\": \"Kegg\", \"x\": -11.517351150512695, \"y\": -16.04863166809082}, {\"pathway\": \"long term depression\", \"Source\": \"Kegg\", \"x\": -28.08815574645996, \"y\": -2.0390102863311768}, {\"pathway\": \"gnrh signaling pathway\", \"Source\": \"Kegg\", \"x\": -0.7892559170722961, \"y\": -19.45136070251465}, {\"pathway\": \"long term potentiation\", \"Source\": \"Kegg\", \"x\": 4.126692771911621, \"y\": 15.45108413696289}, {\"pathway\": \"olfactory transduction\", \"Source\": \"Kegg\", \"x\": -14.804832458496094, \"y\": 3.3798203468322754}, {\"pathway\": \"taste transduction\", \"Source\": \"Kegg\", \"x\": 17.93612289428711, \"y\": -5.827408313751221}, {\"pathway\": \"cell cycle\", \"Source\": \"Kegg\", \"x\": -9.438421249389648, \"y\": -12.741876602172852}, {\"pathway\": \"p53 signaling pathway\", \"Source\": \"Kegg\", \"x\": 16.732500076293945, \"y\": -6.345044136047363}, {\"pathway\": \"ubiquitin mediated proteolysis\", \"Source\": \"Kegg\", \"x\": 9.845611572265625, \"y\": -27.223005294799805}, {\"pathway\": \"tgf beta signaling pathway\", \"Source\": \"Kegg\", \"x\": -13.230647087097168, \"y\": 15.498358726501465}, {\"pathway\": \"antigen processing and presentation\", \"Source\": \"Kegg\", \"x\": 1.947671890258789, \"y\": 14.993699073791504}, {\"pathway\": \"type i diabetes mellitus\", \"Source\": \"Kegg\", \"x\": -9.379870414733887, \"y\": 7.781906604766846}, {\"pathway\": \"cell adhesion molecules\", \"Source\": \"Kegg\", \"x\": 2.4444758892059326, \"y\": -11.317133903503418}, {\"pathway\": \"cytokine cytokine receptor interaction\", \"Source\": \"Kegg\", \"x\": 11.314735412597656, \"y\": -3.156938076019287}, {\"pathway\": \"jak stat signaling pathway\", \"Source\": \"Kegg\", \"x\": 1.252914309501648, \"y\": 16.493207931518555}, {\"pathway\": \"hematopoietic cell lineage\", \"Source\": \"Kegg\", \"x\": 4.220825672149658, \"y\": -4.264805316925049}, {\"pathway\": \"regulation of autophagy\", \"Source\": \"Kegg\", \"x\": -14.193473815917969, \"y\": 18.279436111450195}, {\"pathway\": \"cholera infection\", \"Source\": \"Kegg\", \"x\": -3.9372522830963135, \"y\": -15.895334243774414}, {\"pathway\": \"epithelial cell signaling in helicobacter pylori infection\", \"Source\": \"Kegg\", \"x\": 13.467392921447754, \"y\": -10.351344108581543}, {\"pathway\": \"oxidative phosphorylation\", \"Source\": \"Kegg\", \"x\": 5.72271203994751, \"y\": 10.229391098022461}, {\"pathway\": \"ubiquinone biosynthesis\", \"Source\": \"Kegg\", \"x\": 4.836974143981934, \"y\": -14.628045082092285}, {\"pathway\": \"adipocytokine signaling pathway\", \"Source\": \"Kegg\", \"x\": -15.730012893676758, \"y\": 18.41790008544922}, {\"pathway\": \"ppar signaling pathway\", \"Source\": \"Kegg\", \"x\": -3.3731634616851807, \"y\": -1.7671849727630615}, {\"pathway\": \"fatty acid biosynthesis\", \"Source\": \"Kegg\", \"x\": -15.641959190368652, \"y\": -11.580172538757324}, {\"pathway\": \"polyunsaturated fatty acid biosynthesis\", \"Source\": \"Kegg\", \"x\": -14.410956382751465, \"y\": 1.7896314859390259}, {\"pathway\": \"alzheimers disease\", \"Source\": \"Kegg\", \"x\": -3.3009591102600098, \"y\": 7.8366780281066895}, {\"pathway\": \"dentatorubropallidoluysian atrophy\", \"Source\": \"Kegg\", \"x\": -12.8777437210083, \"y\": 2.022305488586426}, {\"pathway\": \"huntingtons disease\", \"Source\": \"Kegg\", \"x\": -13.92831039428711, \"y\": 16.465639114379883}, {\"pathway\": \"neurodegenerative diseases\", \"Source\": \"Kegg\", \"x\": -12.235395431518555, \"y\": -21.96127700805664}, {\"pathway\": \"notch signaling pathway\", \"Source\": \"Kegg\", \"x\": -7.358044147491455, \"y\": 3.3158926963806152}, {\"pathway\": \"prion disease\", \"Source\": \"Kegg\", \"x\": 18.512542724609375, \"y\": 1.7354145050048828}, {\"pathway\": \"parkinsons disease\", \"Source\": \"Kegg\", \"x\": -13.154515266418457, \"y\": -8.604341506958008}, {\"pathway\": \"amyotrophic lateral sclerosis\", \"Source\": \"Kegg\", \"x\": 0.26976478099823, \"y\": 11.972884178161621}, {\"pathway\": \"lysine biosynthesis\", \"Source\": \"Kegg\", \"x\": 13.784589767456055, \"y\": 23.91696548461914}, {\"pathway\": \"maturity onset diabetes of the young\", \"Source\": \"Kegg\", \"x\": 25.974414825439453, \"y\": 2.11128306388855}, {\"pathway\": \"complement and coagulation cascades\", \"Source\": \"Kegg\", \"x\": -1.7176607847213745, \"y\": -24.171045303344727}, {\"pathway\": \"renin angiotensin system\", \"Source\": \"Kegg\", \"x\": 12.735748291015625, \"y\": 21.700542449951172}, {\"pathway\": \"1 and 2 methylnaphthalene degradation\", \"Source\": \"Kegg\", \"x\": 18.766521453857422, \"y\": 15.45506763458252}, {\"pathway\": \"ethylbenzene degradation\", \"Source\": \"Kegg\", \"x\": -13.711762428283691, \"y\": -4.940710544586182}, {\"pathway\": \"benzoate degradation via coa ligation\", \"Source\": \"Kegg\", \"x\": -0.7833523750305176, \"y\": 20.916080474853516}, {\"pathway\": \"limonene and pinene degradation\", \"Source\": \"Kegg\", \"x\": 11.863412857055664, \"y\": -9.453818321228027}, {\"pathway\": \"alkaloid biosynthesis ii\", \"Source\": \"Kegg\", \"x\": 24.926942825317383, \"y\": -10.457541465759277}, {\"pathway\": \"phenylalanine metabolism\", \"Source\": \"Kegg\", \"x\": 12.027355194091797, \"y\": 11.850410461425781}, {\"pathway\": \"tyrosine metabolism\", \"Source\": \"Kegg\", \"x\": -17.024065017700195, \"y\": -6.830172061920166}, {\"pathway\": \"aminophosphonate metabolism\", \"Source\": \"Kegg\", \"x\": -20.47085189819336, \"y\": 9.179293632507324}, {\"pathway\": \"naphthalene and anthracene degradation\", \"Source\": \"Kegg\", \"x\": 5.307245254516602, \"y\": -1.5374306440353394}, {\"pathway\": \"selenoamino acid metabolism\", \"Source\": \"Kegg\", \"x\": 21.43680191040039, \"y\": 10.039520263671875}, {\"pathway\": \"histidine metabolism\", \"Source\": \"Kegg\", \"x\": 14.027450561523438, \"y\": 12.8308687210083}, {\"pathway\": \"tryptophan metabolism\", \"Source\": \"Kegg\", \"x\": 6.185763835906982, \"y\": 7.7388739585876465}, {\"pathway\": \"3 chloroacrylic acid degradation\", \"Source\": \"Kegg\", \"x\": -4.889517307281494, \"y\": -26.93632698059082}, {\"pathway\": \"bile acid biosynthesis\", \"Source\": \"Kegg\", \"x\": 11.06396484375, \"y\": 19.444265365600586}, {\"pathway\": \"fatty acid metabolism\", \"Source\": \"Kegg\", \"x\": -3.2364304065704346, \"y\": -11.813963890075684}, {\"pathway\": \"glycerolipid metabolism\", \"Source\": \"Kegg\", \"x\": 9.683574676513672, \"y\": -21.022193908691406}, {\"pathway\": \"ascorbate and aldarate metabolism\", \"Source\": \"Kegg\", \"x\": -14.696240425109863, \"y\": -4.381252765655518}, {\"pathway\": \"beta alanine metabolism\", \"Source\": \"Kegg\", \"x\": 7.887968063354492, \"y\": -14.592414855957031}, {\"pathway\": \"urea cycle and metabolism of amino groups\", \"Source\": \"Kegg\", \"x\": -21.772865295410156, \"y\": -6.039887428283691}, {\"pathway\": \"butanoate metabolism\", \"Source\": \"Kegg\", \"x\": -18.532114028930664, \"y\": -26.048646926879883}, {\"pathway\": \"lysine degradation\", \"Source\": \"Kegg\", \"x\": 8.55601692199707, \"y\": -18.453575134277344}, {\"pathway\": \"propanoate metabolism\", \"Source\": \"Kegg\", \"x\": -5.072546005249023, \"y\": -27.452224731445312}, {\"pathway\": \"valine leucine and isoleucine degradation\", \"Source\": \"Kegg\", \"x\": -3.434279441833496, \"y\": 20.24922752380371}, {\"pathway\": \"glycolysis and gluconeogenesis\", \"Source\": \"Kegg\", \"x\": 23.70863914489746, \"y\": -10.760626792907715}, {\"pathway\": \"pyruvate metabolism\", \"Source\": \"Kegg\", \"x\": -2.5872809886932373, \"y\": 28.672443389892578}, {\"pathway\": \"caprolactam degradation\", \"Source\": \"Kegg\", \"x\": -21.1705265045166, \"y\": -5.493078231811523}, {\"pathway\": \"fatty acid elongation in mitochondria\", \"Source\": \"Kegg\", \"x\": 6.309348106384277, \"y\": -13.100468635559082}, {\"pathway\": \"synthesis and degradation of ketone bodies\", \"Source\": \"Kegg\", \"x\": -5.350968837738037, \"y\": -6.033792018890381}, {\"pathway\": \"1,4 dichlorobenzene degradation\", \"Source\": \"Kegg\", \"x\": 19.815237045288086, \"y\": 21.820112228393555}, {\"pathway\": \"gamma hexachlorocyclohexane degradation\", \"Source\": \"Kegg\", \"x\": 19.90184783935547, \"y\": 19.685693740844727}, {\"pathway\": \"aminosugars metabolism\", \"Source\": \"Kegg\", \"x\": -18.43570327758789, \"y\": -15.849452018737793}, {\"pathway\": \"riboflavin metabolism\", \"Source\": \"Kegg\", \"x\": -25.644784927368164, \"y\": -4.909708499908447}, {\"pathway\": \"thiamine metabolism\", \"Source\": \"Kegg\", \"x\": -25.450559616088867, \"y\": -3.1752679347991943}, {\"pathway\": \"bisphenol a degradation\", \"Source\": \"Kegg\", \"x\": -15.525940895080566, \"y\": -27.114870071411133}, {\"pathway\": \"tetrachloroethene degradation\", \"Source\": \"Kegg\", \"x\": -6.589664936065674, \"y\": 8.205245971679688}, {\"pathway\": \"fructose and mannose metabolism\", \"Source\": \"Kegg\", \"x\": 9.155430793762207, \"y\": -3.3921804428100586}, {\"pathway\": \"galactose metabolism\", \"Source\": \"Kegg\", \"x\": 6.950483322143555, \"y\": -25.341693878173828}, {\"pathway\": \"glycine serine and threonine metabolism\", \"Source\": \"Kegg\", \"x\": 0.8592827320098877, \"y\": -10.96346664428711}, {\"pathway\": \"nucleotide sugars metabolism\", \"Source\": \"Kegg\", \"x\": -13.390896797180176, \"y\": 18.307546615600586}, {\"pathway\": \"streptomycin biosynthesis\", \"Source\": \"Kegg\", \"x\": -9.851150512695312, \"y\": 23.182462692260742}, {\"pathway\": \"alpha linolenic acid metabolism\", \"Source\": \"Kegg\", \"x\": 12.378458023071289, \"y\": 3.1633541584014893}, {\"pathway\": \"ether lipid metabolism\", \"Source\": \"Kegg\", \"x\": 24.89794921875, \"y\": -4.597784519195557}, {\"pathway\": \"glycerophospholipid metabolism\", \"Source\": \"Kegg\", \"x\": -3.2104220390319824, \"y\": 27.263145446777344}, {\"pathway\": \"arachidonic acid metabolism\", \"Source\": \"Kegg\", \"x\": -22.847440719604492, \"y\": -11.146718978881836}, {\"pathway\": \"linoleic acid metabolism\", \"Source\": \"Kegg\", \"x\": 14.955755233764648, \"y\": 16.911331176757812}, {\"pathway\": \"monoterpenoid biosynthesis\", \"Source\": \"Kegg\", \"x\": 14.125316619873047, \"y\": -10.783281326293945}, {\"pathway\": \"androgen and estrogen metabolism\", \"Source\": \"Kegg\", \"x\": 0.8011097311973572, \"y\": -19.191749572753906}, {\"pathway\": \"pentose and glucuronate interconversions\", \"Source\": \"Kegg\", \"x\": -15.046473503112793, \"y\": 1.082504153251648}, {\"pathway\": \"porphyrin and chlorophyll metabolism\", \"Source\": \"Kegg\", \"x\": -18.50104331970215, \"y\": -18.013259887695312}, {\"pathway\": \"glutathione metabolism\", \"Source\": \"Kegg\", \"x\": 11.497472763061523, \"y\": -13.39468002319336}, {\"pathway\": \"metabolism of xenobiotics by cytochrome p450\", \"Source\": \"Kegg\", \"x\": -28.371633529663086, \"y\": -6.451005458831787}, {\"pathway\": \"folate biosynthesis\", \"Source\": \"Kegg\", \"x\": 6.765783786773682, \"y\": 25.495092391967773}, {\"pathway\": \"starch and sucrose metabolism\", \"Source\": \"Kegg\", \"x\": 1.357383370399475, \"y\": 0.9427309632301331}, {\"pathway\": \"c21 steroid hormone metabolism\", \"Source\": \"Kegg\", \"x\": 17.440818786621094, \"y\": 7.79496955871582}, {\"pathway\": \"styrene degradation\", \"Source\": \"Kegg\", \"x\": -16.24764060974121, \"y\": -26.670846939086914}, {\"pathway\": \"cyanoamino acid metabolism\", \"Source\": \"Kegg\", \"x\": 15.444511413574219, \"y\": -21.506410598754883}, {\"pathway\": \"methane metabolism\", \"Source\": \"Kegg\", \"x\": 5.599283695220947, \"y\": 15.025264739990234}, {\"pathway\": \"phenylpropanoid biosynthesis\", \"Source\": \"Kegg\", \"x\": 15.650359153747559, \"y\": -10.526273727416992}, {\"pathway\": \"one carbon pool by folate\", \"Source\": \"Kegg\", \"x\": 22.68391990661621, \"y\": 5.300788879394531}, {\"pathway\": \"taurine and hypotaurine metabolism\", \"Source\": \"Kegg\", \"x\": -14.333499908447266, \"y\": -27.242971420288086}, {\"pathway\": \"alanine and aspartate metabolism\", \"Source\": \"Kegg\", \"x\": 8.881205558776855, \"y\": -28.72945213317871}, {\"pathway\": \"aminoacyl trna biosynthesis\", \"Source\": \"Kegg\", \"x\": -17.889196395874023, \"y\": -0.2625240683555603}, {\"pathway\": \"valine leucine and isoleucine biosynthesis\", \"Source\": \"Kegg\", \"x\": 1.289156436920166, \"y\": 4.157289981842041}, {\"pathway\": \"c5 branched dibasic acid metabolism\", \"Source\": \"Kegg\", \"x\": 3.520587921142578, \"y\": -21.732784271240234}, {\"pathway\": \"pantothenate and coa biosynthesis\", \"Source\": \"Kegg\", \"x\": 9.466812133789062, \"y\": -26.948392868041992}, {\"pathway\": \"citrate cycle\", \"Source\": \"Kegg\", \"x\": -12.967565536499023, \"y\": -16.397071838378906}, {\"pathway\": \"reductive carboxylate cycle\", \"Source\": \"Kegg\", \"x\": -28.711214065551758, \"y\": 4.162548542022705}, {\"pathway\": \"glyoxylate and dicarboxylate metabolism\", \"Source\": \"Kegg\", \"x\": -10.943666458129883, \"y\": -5.752692699432373}, {\"pathway\": \"alkaloid biosynthesis i\", \"Source\": \"Kegg\", \"x\": 0.2902225852012634, \"y\": -10.982032775878906}, {\"pathway\": \"novobiocin biosynthesis\", \"Source\": \"Kegg\", \"x\": -4.565511703491211, \"y\": 28.014019012451172}, {\"pathway\": \"phenylalanine tyrosine and tryptophan biosynthesis\", \"Source\": \"Kegg\", \"x\": -6.286781311035156, \"y\": -25.156736373901367}, {\"pathway\": \"cysteine metabolism\", \"Source\": \"Kegg\", \"x\": 13.776561737060547, \"y\": -9.451543807983398}, {\"pathway\": \"methionine metabolism\", \"Source\": \"Kegg\", \"x\": -7.359382629394531, \"y\": 11.705804824829102}, {\"pathway\": \"carbon fixation\", \"Source\": \"Kegg\", \"x\": 6.6805644035339355, \"y\": 8.515531539916992}, {\"pathway\": \"pentose phosphate pathway\", \"Source\": \"Kegg\", \"x\": 9.901883125305176, \"y\": 20.103458404541016}, {\"pathway\": \"inositol metabolism\", \"Source\": \"Kegg\", \"x\": 14.495559692382812, \"y\": -21.872121810913086}, {\"pathway\": \"arginine and proline metabolism\", \"Source\": \"Kegg\", \"x\": -18.20347785949707, \"y\": -9.675378799438477}, {\"pathway\": \"d arginine and d ornithine metabolism\", \"Source\": \"Kegg\", \"x\": 6.445102691650391, \"y\": 12.271345138549805}, {\"pathway\": \"d glutamine and d glutamate metabolism\", \"Source\": \"Kegg\", \"x\": -3.4090890884399414, \"y\": 8.787223815917969}, {\"pathway\": \"nitrogen metabolism\", \"Source\": \"Kegg\", \"x\": -14.091941833496094, \"y\": 11.42345142364502}, {\"pathway\": \"glutamate metabolism\", \"Source\": \"Kegg\", \"x\": 12.568406105041504, \"y\": 23.029754638671875}, {\"pathway\": \"peptidoglycan biosynthesis\", \"Source\": \"Kegg\", \"x\": -23.713558197021484, \"y\": -8.173540115356445}, {\"pathway\": \"caffeine metabolism\", \"Source\": \"Kegg\", \"x\": -12.198799133300781, \"y\": 19.29998207092285}, {\"pathway\": \"dna polymerase\", \"Source\": \"Kegg\", \"x\": -17.364585876464844, \"y\": 18.160133361816406}, {\"pathway\": \"purine metabolism\", \"Source\": \"Kegg\", \"x\": -12.239782333374023, \"y\": 18.226070404052734}, {\"pathway\": \"pyrimidine metabolism\", \"Source\": \"Kegg\", \"x\": -20.74567985534668, \"y\": 6.949365615844727}, {\"pathway\": \"rna polymerase\", \"Source\": \"Kegg\", \"x\": -7.87260103225708, \"y\": -25.277284622192383}, {\"pathway\": \"nicotinate and nicotinamide metabolism\", \"Source\": \"Kegg\", \"x\": 14.568680763244629, \"y\": -1.735870599746704}, {\"pathway\": \"vitamin b6 metabolism\", \"Source\": \"Kegg\", \"x\": -18.264928817749023, \"y\": -12.696812629699707}, {\"pathway\": \"chondroitin sulfate biosynthesis\", \"Source\": \"Kegg\", \"x\": 6.603456974029541, \"y\": 18.275367736816406}, {\"pathway\": \"sulfur metabolism\", \"Source\": \"Kegg\", \"x\": -2.2547500133514404, \"y\": 11.687116622924805}, {\"pathway\": \"glycan structures biosynthesis 1\", \"Source\": \"Kegg\", \"x\": -18.895477294921875, \"y\": -11.051804542541504}, {\"pathway\": \"n glycan biosynthesis\", \"Source\": \"Kegg\", \"x\": 11.535344123840332, \"y\": 15.764122009277344}, {\"pathway\": \"keratan sulfate biosynthesis\", \"Source\": \"Kegg\", \"x\": 5.815224647521973, \"y\": -16.578514099121094}, {\"pathway\": \"o glycan biosynthesis\", \"Source\": \"Kegg\", \"x\": -22.19713020324707, \"y\": 3.4086191654205322}, {\"pathway\": \"heparan sulfate biosynthesis\", \"Source\": \"Kegg\", \"x\": 10.920904159545898, \"y\": -4.9131646156311035}, {\"pathway\": \"glycan structures biosynthesis 2\", \"Source\": \"Kegg\", \"x\": 22.746601104736328, \"y\": -8.583183288574219}, {\"pathway\": \"glycosphingolipid biosynthesis neo lactoseries\", \"Source\": \"Kegg\", \"x\": 7.823408126831055, \"y\": -9.779397010803223}, {\"pathway\": \"glycosphingolipid biosynthesis lactoseries\", \"Source\": \"Kegg\", \"x\": 10.46964168548584, \"y\": -13.506150245666504}, {\"pathway\": \"glycosphingolipid biosynthesis ganglioseries\", \"Source\": \"Kegg\", \"x\": 26.21002960205078, \"y\": 0.6284250617027283}, {\"pathway\": \"glycosphingolipid biosynthesis globoseries\", \"Source\": \"Kegg\", \"x\": -9.510930061340332, \"y\": 19.04582977294922}, {\"pathway\": \"glycosylphosphatidylinositol anchor biosynthesis\", \"Source\": \"Kegg\", \"x\": 4.118001937866211, \"y\": 12.360539436340332}, {\"pathway\": \"glycan structures degradation\", \"Source\": \"Kegg\", \"x\": 24.433059692382812, \"y\": -12.93445110321045}, {\"pathway\": \"glycosaminoglycan degradation\", \"Source\": \"Kegg\", \"x\": -12.321638107299805, \"y\": 4.993157386779785}, {\"pathway\": \"n glycan degradation\", \"Source\": \"Kegg\", \"x\": 8.98360538482666, \"y\": -6.195254325866699}, {\"pathway\": \"sphingolipid metabolism\", \"Source\": \"Kegg\", \"x\": -24.820621490478516, \"y\": -11.68470573425293}, {\"pathway\": \"biotin metabolism\", \"Source\": \"Kegg\", \"x\": -11.515620231628418, \"y\": 22.51636505126953}, {\"pathway\": \"circadian rhythm\", \"Source\": \"Kegg\", \"x\": 2.9428422451019287, \"y\": -3.1617109775543213}, {\"pathway\": \"ribosome\", \"Source\": \"Kegg\", \"x\": -22.8239688873291, \"y\": -9.919317245483398}, {\"pathway\": \"abc transporters general\", \"Source\": \"Kegg\", \"x\": 18.707317352294922, \"y\": -8.956354141235352}, {\"pathway\": \"protein export\", \"Source\": \"Kegg\", \"x\": -11.714150428771973, \"y\": 12.417131423950195}, {\"pathway\": \"biosynthesis of steroids\", \"Source\": \"Kegg\", \"x\": 7.072144508361816, \"y\": 0.01632099039852619}, {\"pathway\": \"terpenoid biosynthesis\", \"Source\": \"Kegg\", \"x\": -1.0852041244506836, \"y\": 23.610475540161133}, {\"pathway\": \"atrazine degradation\", \"Source\": \"Kegg\", \"x\": -1.3915777206420898, \"y\": 29.24065399169922}, {\"pathway\": \"basal transcription factors\", \"Source\": \"Kegg\", \"x\": 25.89386749267578, \"y\": -9.177733421325684}, {\"pathway\": \"snare interactions in vesicular transport\", \"Source\": \"Kegg\", \"x\": -10.576642990112305, \"y\": -15.496538162231445}, {\"pathway\": \"retinol metabolism\", \"Source\": \"Kegg\", \"x\": -16.05585289001465, \"y\": -6.449162483215332}, {\"pathway\": \"lipoic acid metabolism\", \"Source\": \"Kegg\", \"x\": 5.300985336303711, \"y\": -15.323957443237305}, {\"pathway\": \"proteasome\", \"Source\": \"Kegg\", \"x\": -14.270844459533691, \"y\": 21.203001022338867}]}}, {\"mode\": \"vega-lite\"});\n",
       "</script>"
      ],
      "text/plain": [
       "alt.Chart(...)"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# df\n",
    "alt.Chart(df).mark_circle(size=50).encode(\n",
    "    x='x',\n",
    "    y='y',\n",
    "    color='Source',\n",
    "    tooltip=['pathway', 'Source', 'x','y']\n",
    ").properties(width=600, height=400)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
